<p>Suspension cultures of EML cells were maintained in SCF containing medium as previously described <xref ref-type="bibr" rid="pgen.1002565-Ye1">[13]</xref>. Total EML cells were washed twice in FACS buffer (0.5% BSA, 1 mM EDTA, 1&#215; PBS) and resuspended in 40 &#181;l FACS buffer per 1&#215;10<sup>7</sup> cells. 15 &#181;l of Mouse Lineage Depletion Cocktail biotin conjugated antibody (Miltenyi Biotec) was added to the cells for 20 minutes at 4&#176;C. The labeled cells were washed twice in FACS buffer and resuspended in 80 &#181;l FACS buffer. 20 &#181;l of paramagnetic microbeads conjugated to anti-biotin antibody (Miltenyi Biotec) was added to the cells and incubated for 20 minutes at 4&#176;C. The labeled cells were washed twice and resuspended in 2 ml FACS buffer per 1&#215;10<sup>7</sup> cells. The cells were separated twice in an AutoMACS cell separator (Miltenyi Biotec) using the depletion program (0.5 mls per minute). The lineage negative (Lin-) fraction was resuspended in 100 &#181;l (per 1&#215;10<sup>7</sup> cells) of FACS buffer and CD34 biotin conjugated antibody was added (1 &#181;g per 1&#215;10<sup>6</sup> cells). The labeled cells were washed in FACS buffer as above and bound to anti-biotin coated beads. The cells were separated in an AutoMACS cell separator (Miltenyi Biotec) using a double positive separation program. The subsequent Lin-CD34&#8722; fraction was resorted with the AutoMACS separator using the depletion program. Lin-CD34+ and Lin-CD34+ cells were collected. The cell purity was checked after each separation using FACS and only cell purity &gt;90% was used for further experimentation.</p><p>Total EML cells were washed twice in FACS buffer (0.5% BSA, 1 mM EDTA, 1&#215; PBS) and resuspended in 100 &#181;l FACS buffer per 1&#215;10<sup>6</sup> cells. CD34-FITC (1 &#181;g per 1&#215;10<sup>6</sup> cells; Ebiosciences) was added to the cells and incubated for 1 hour at 4&#176;C. Sca1-PE (0.06 &#181;g per 1&#215;10<sup>6</sup> cells; Ebiosciences) and Lineage Cocktail APC (5 &#181;l per 1&#215;10<sup>6</sup> cells; Miltenyi Biotec) were added to the cells and incubated for an additional 30 minutes. Lin-SCA+CD34+ and Lin-SCA-CD34&#8722; cells were collected using FACS Aria (Beckman).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">mRNA samples were prepared from 2&#215;10<sup>6</sup> CD34+ and CD34&#8722; cells. RNA-Seq was performed as described <xref ref-type="bibr" rid="pgen.1002565-Wu1">[50]</xref>. Two biological replicas and two technical replicas were used for each cell type. The mouse genome sequence, annotation and genomic features (genes, cDNAs, 3&#8242; UTRs, 5&#8242; UTRs, introns, exons, intergenic regions, ESTs) for the mm9 database release were directly downloaded from UCSC Table Browser (<ext-link ext-link-type="uri" ns0:href="http://genome.ucsc.edu">http://genome.ucsc.edu</ext-link>) or obtained from Galaxy (<ext-link ext-link-type="uri" ns0:href="http://galaxy.psu.edu/">http://galaxy.psu.edu/</ext-link>). Raw Illumina reads were obtained after base calling in the Solexa Pipeline version 0.2.2.6. RNA-Seq reads were mapped to the mouse genome using Illumina's ELAND software. Differentially expressed gene features were identified using the ERANGE package <xref ref-type="bibr" rid="pgen.1002565-Mortazavi1">[51]</xref>. Read coverage along the annotated transcription units was calculated using the ShortRead package <xref ref-type="bibr" rid="pgen.1002565-Morgan1">[52]</xref>.</p><p>Repetitive mapped reads were combined with uniquely mapped reads to produce a final RPKM (reads per kilobase of mRNA, per million total reads), using the procedure defined for ERANGE, by calculating the probability that a multiread came from a particular known or candidate exon based on the distribution of counts of uniquely mapped reads in each exon. The resulting fractional counts were added to the total count for the gene locus, which was renormalized into a multi RPKM (Gene expression values: <xref ref-type="supplementary-material" rid="pgen.1002565.s008">Dataset S1</xref>).</p><p>Transcription factors were identified from the list provided by Luscombe et al <xref ref-type="bibr" rid="pgen.1002565-Vaquerizas1">[53]</xref> of human transcription factors. Homologous mouse genes were obtained for each from the Ensembl database using the biomaRt package. We identified genes with different expression levels in CD34+ versus CD34&#8722; cells as those with at least a two fold difference in RPKM and in which both cell types had a minimum of 2 RPKM.</p><p>ChIP-Seq was performed as described <xref ref-type="bibr" rid="pgen.1002565-Hartman1">[54]</xref>, <xref ref-type="bibr" rid="pgen.1002565-Euskirchen1">[55]</xref>, <xref ref-type="bibr" rid="pgen.1002565-Raha1">[56]</xref>. 5&#215;10<sup>7</sup> formaldehyde cross-linked Lin-CD34+ and Lin-CD34&#8722; cells were used. TCF7 goat polyclonal antibody TCF1(H18) (Santa Cruz Biotechnology: catalog#SC8589), anti-AML(RUNX1) rabbit polyclonal antibody <xref ref-type="bibr" rid="pgen.1002565-Hug1">[36]</xref> (CalBiochem: catalog#PC284), and monoclonal RNA PolII antibody (Covance: catalog#8WG16-MMS-126R) were used. IP-western experiments were done to ensure the specificity of the antibodies.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">ChIP-Sequencing data were analyzed using the PeakSeq program as previously described <xref ref-type="bibr" rid="pgen.1002565-Rozowsky1">[57]</xref>. The transcription factor binding loci were extracted with statistically significant signals (q-value&lt;0.001). Subsequently, we mapped the binding sites to RefSeq genes in UCSC mm9 database (genome.ucsc.edu). A gene was designated as the target gene if the peak was present within 3000 nt upstream of the transcription start site or inside of the gene (including both exonic and intronic regions). ChIP-Seq data have been deposited to GEO database (GSE31221; reviewer access link: <ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?token=lxglzumiewawcha&amp;acc=GSE31221">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?token=lxglzumiewawcha&amp;acc=GSE31221</ext-link>.) In TCF7 and RUNX1 coregulation analysis, known motifs for each of TCF7 and RUNX1 were obtained from Jaspar and Transfac databases. We random selected 600 sequences bound in TCF7 and RUNX1 ChIP-Seq experiments, and performed de novo binding motif search. We compared these de novo binding motifs with the known motifs (from Jaspar and Transfac databases) of RUNX1 and TCF7.</p><p>We used BiNGO <xref ref-type="bibr" rid="pgen.1002565-Maere1">[35]</xref> to determine the statistically over-represented (p-value&lt;0.0001) Gene Ontology (GO) categories within the target gene sets of the transcription factors, and then visualized the relationships of these GO categories with the Cytoscape software <xref ref-type="bibr" rid="pgen.1002565-Shannon1">[58]</xref>.</p><p>We also performed Gene Set Enrichment Analysis (GSEA) <xref ref-type="bibr" rid="pgen.1002565-Subramanian1">[37]</xref> to correlate transcription factor binding information with the gene expression data. The expression dataset was rank-ordered by fold change (difference of classes ranking metric) such that the most up-regulated genes in Lin-CD34+ cells were on the top of the ranked list, while the most up-regulated genes in Lin-CD34&#8722; cells (down-regulated genes in Lin-CD34+ cells) were at the bottom of the ranked list. GSEA analysis was used to determine whether members of a binding target list, are randomly distributed throughout the gene expression data or primarily enriched toward the top or bottom of the gene expression list using the default weighted enrichment statistic.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Ingenuity Pathway Analysis (IPA) was performed to display the transcription factor regulatory networks (<ext-link ext-link-type="uri" ns0:href="http://www.ingenuity.com/">http://www.ingenuity.com/</ext-link>).</p><p>Fresh EML cells were recovered one week prior to shRNA experiments in SCF containing growth medium (GM) as previously described <xref ref-type="bibr" rid="pgen.1002565-Ye1">[13]</xref>. 1.5&#215;10<sup>4</sup> EML Lin-CD34+cells were double FACS sorted and infected with shRNA constructs containing <italic>Tcf7</italic> (Sigma Aldrich) at an MOI&#8202;=&#8202;2 in a round bottom 96 well plate. Five shRNA constructs targeting different regions of the <italic>Tcf7</italic> gene were used (Sigma, shRNA product numbers: TRCN0000012678, TRCN0000012679, TRCN0000012680, TRCN0000012681, TRCN0000012682). SHC002VMISSION Non-Target shRNA Control Transduction Particles (Sigma) were used as shRNA negative control. To increase transduction efficiency the ExpressMag systems (Sigma Aldrich) was used according to manufacturer's instructions. 24 hours post transduction, cells were selected in EML GM containing puromycin (1.2 &#181;g/ml). When indicated, selected cells were grown in expansion medium (IMDB, 20% heat inactivated horse serum, 100 ng/ml SCF [PeproTech]). Cells were analyzed for knockdown efficiency using qPCR (<italic>Tcf7</italic> forward primer sequence <named-content content-type="gene">ATCCTTGATGCTGGGATTCTG</named-content>; <italic>Tcf7</italic> reverse primer sequence <named-content content-type="gene">CTTCTCTTGCCTTGGGTTCTG</named-content>. CD34 forward primer sequence <named-content content-type="gene">aggctgatgctggtgctagt</named-content>; reverse primer sequence <named-content content-type="gene">ccccagctttctcaagtcag</named-content>. Two internal controls: HPRT forward primer <named-content content-type="gene">tatgccgaggatttggaaaa</named-content>; HPRT reverse primer <named-content content-type="gene">acagagggccacaatgtgat</named-content>, and/or beta Actin forward primer <named-content content-type="gene">gatctggcaccacaccttct</named-content>; reverse primer <named-content content-type="gene">accagaggcatacagggaca</named-content>). In addition, Western blot analysis was performed on puromycin-selected <italic>Tcf7</italic> knockdown cell lines to examine TCF7 and RUNX1 protein expression. The polyclonal TCF7 antibody (Sigma Aldrich, catalog#AV34782) and RUNX1 antibody (Abcam, catalog# ab23980) were used. The anti actin (Abcam, catalog#ab8229) antibody was used to indicate equal loading.</p><p>For Illumina array analysis, Lin- CD34+cells (1&#215;10<sup>5</sup>) cells were transduced at an MOI&#8202;=&#8202;1 with a <italic>Tcf7</italic> targeting shRNA construct TRCN0000012679 or a shRNA negative control. After a 24-hour incubation with the shRNA-containing virus, the cells were grown in EML GM for 24 hours, then cells were selected for 24 hours in puromycin (1.2 &#181;g/ml). Cells were harvested (a total of four days after initial sort) and total RNA extracted. Total RNA from <italic>Tcf7</italic> shRNA knockdown cells and control cells (transfected with scrambled shRNA) was purified using the RNeasy Plus kit from Qiagen. Hybridization to Illumina Mouse WG-6 v2.0 Expression BeadChips was conducted at the Stanford Functional Genomics Facility using standard Illumina protocols. The microarray data was processed using the R version 2.11 Bioconductor Lumi package. The gene expression values were offset so that all values were made positive, subjected to the VST variance stabilization transformation, and were then quantile normalized. Z scores are plotted where Z&#8202;=&#8202;(x&#8722;&#956;)/&#963;, x is the log2 transformed gene expression measurement and &#956; and &#963; are the mean and standard deviations of expression of the gene. The microarray data is in compliance with MIAME guidelines. The data have been deposited in GEO database (GSE30068).</p><p>To test for the presence of <italic>wnt10a</italic> mRNA, four sets of PCR primers were designed cross introns, which could distinguish between genomic DNA and spliced cDNA.</p><p>Wnt10a forward primer 1:<named-content content-type="gene">GCGCTCCTGTTCTTCCTACT</named-content>, Wnt10a reverse primer 1: <named-content content-type="gene">GATCTGGATGCCCTGGATAG</named-content>; Wnt10a forward primer 2: <named-content content-type="gene">GGCGCTCCTGTTCTTCCTAC</named-content>, Wnt10a reverse primer 2: <named-content content-type="gene">ATGCCCTGGATAGCAGAGG</named-content>; Wnt10a forward primer 3: <named-content content-type="gene">CATGAGTGCCAGCATCAGTT</named-content>, Wnt10a reverse primer 3: <named-content content-type="gene">ACCGCAAGCCTTCAGTTTAC</named-content>; Wnt10a forward primer 4: <named-content content-type="gene">CATGAGTGCCAGCATCAGTT</named-content>, Wnt10a reverse primer 4: <named-content content-type="gene">AGCCTTCAGTTTACCCAGAGC</named-content>.</p><p>Total EML cells (5&#215;10<sup>4</sup>) were infected with lentivirus containing either a CMV-GFP construct or a <italic>Tcf/Lef</italic>-GFP construct (SABiosciences) at a MOI of 1, 5, 10, and 20. After 24 hours, cells were selected in EML GM with puromycin (1.2 &#181;g/ml). Selected cells were expanded in EML GM for 4&#8211;6 days. Cells were incubated for 24 hours in either LiCL (50 mM), WNT3a (400 uM, PeproTech) or WNT5a (400 um, PeproTech). Cells were analyzed by FACSCalibur (BD Biosciences).</p>