<p>
<italic>S. oneidensis</italic> MR-1 was purchased from ATCC (catalog number 700550). <italic>E. coli</italic> conjugation donor strain WM3064 was a gift of William Metcalf (U. of Illinois). The <italic>E. coli</italic> &#916;<italic>argE</italic> strain was obtained from the KEIO collection <xref ref-type="bibr" rid="pgen.1002385-Baba1">[11]</xref>. See <xref ref-type="supplementary-material" rid="pgen.1002385.s012">Table S4</xref> for the strains used in this study. All strains were commonly cultured in Luria-Bertani broth (LB) with appropriate antibiotic selection; kanamycin (50 &#181;g/ml) for MR-1 transposon mutants and for the <italic>E. coli &#916;argE</italic> strain, and gentamicin (15 &#181;g/ml) for complementation strains containing plasmid pBBR1-MCS5. To grow the diaminopimelic acid (dap) auxotroph WM3064, dap was added to the media at a final concentration of 300 &#181;M. Our standard MR-1 minimal media contained salts (per liter: 1.5 g NH<sub>4</sub>Cl, 0.1 g KCl, 1.75 g NaCl, 0.61 g MgCl<sub>2</sub>-6H<sub>2</sub>0, 0.6 g NaH<sub>2</sub>PO<sub>4</sub>), 30 mM PIPES buffer, 20 mM DL-lactate, Wolfe's vitamins, and Wolfe's minerals. For anaerobic minimal media, we added one of the following electron acceptors: fumarate (30 mM), dimethyl sulfoxide (20 mM), iron (III) citrate (10 mM), manganese (IV) oxide (30 mM), trimethylamine N-oxide (10 mM), nitrate (5 mM), or cobalt (III)-EDTA (5 mM). For anaerobic experiments, manganese oxide <xref ref-type="bibr" rid="pgen.1002385-Burdige1">[72]</xref> and cobalt (III)-EDTA <xref ref-type="bibr" rid="pgen.1002385-Hau1">[73]</xref> were prepared as described. For pool experiments with alternative nutrient sources, we replaced the DL-lactate with a different carbon source, the NH<sub>4</sub>Cl with a different nitrogen source, both with a single carbon/nitrogen source, or the NaH<sub>2</sub>PO<sub>4</sub> with an alternative phosphorous source. For alternative sulfur sources, we replaced all sulfate containing minerals in the Wolfe's mineral mixture with non-sulfur containing versions such that the added sulfur source served as the sole source. MR-1 was typically grown at 30&#176;C; <italic>E. coli</italic> was grown at 37&#176;C.</p><p>We previously reported the generation and preliminary analysis of a library of 7,387 transposon insertion mutants in MR-1 <xref ref-type="bibr" rid="pgen.1002385-Oh2">[19]</xref>. To achieve greater coverage of the genome, we mapped an additional 17,301 mutants using a two-step arbitrary PCR and sequencing protocol, as described <xref ref-type="bibr" rid="pgen.1002385-Oh2">[19]</xref>. Briefly, each TagModule contains two unique 20 bp DNA sequences, the uptag and downtag, each flanked by common PCR priming sites. The TagModules are cloned into a Gateway entry vector and can be readily transferred to any Gateway compatible destination vector via the LR clonase reaction (Invitrogen). We transferred the TagModules into two different transposon vectors that are active in MR-1, the <italic>Tn</italic>5-based pRL27 <xref ref-type="bibr" rid="pgen.1002385-Larsen1">[74]</xref> and <italic>mariner</italic>-based pMiniHimar_RB1 <xref ref-type="bibr" rid="pgen.1002385-Bouhenni1">[75]</xref>. The tagged transposons were introduced into MR-1 by conjugation with an <italic>E. coli</italic> WM3064 donor strain carrying the appropriate suicide vector. We used a two-step arbitrary PCR and sequencing protocol to simultaneously map the transposon insertion location and identify the TagModule. All mutants were stored as glycerol stocks in either 96-well or 384-well plates. Overall, we observe transposon insertion biases both on the main 5 MB chromosome and on the 161 kB megaplasmid (<xref ref-type="supplementary-material" rid="pgen.1002385.s006">Figure S6</xref>). Analysis of the insertion preferences for <italic>Tn</italic>5 and <italic>mariner</italic> indicates that their insertion biases are not equal (<xref ref-type="supplementary-material" rid="pgen.1002385.s007">Figure S7</xref>), thus illustrating the benefit of using multiple transposons to achieve maximal coverage. Additionally, we observed a greater than 4-fold increase in mapped megaplasmid insertions relative to the expectation (based on size). Given that the megaplasmid appears to be equal in copy number to the main chromosome (data not shown), we speculate that the megaplasmid is more accessible to transposon mutagenesis. The total MR-1 transposon collection consists of 24,688 archived strains and represents mutants in 3,447 unique genes (<xref ref-type="supplementary-material" rid="pgen.1002385.s009">Table S1</xref> for full list).</p><p>We constructed two mutant pools, the upPool and dnPool, by mixing equal volumes of overnight LB cultures for each strain. These pools were designed such that each strain's TagModule is unique within that pool and to achieve maximal coverage of the genome. The upPool contains 4,058 strains whose tags are detected at a threshold &#8764;5&#215; over background in representative start hybridizations. Conversely, the dnPool has 3,977 strains that meet the same detection criteria. A total of 165 strains across both pools (&#8764;2% of the overall number of strains that we attempted to pool) were not detected at 5&#215; over background and fitness values were not calculated for these strains. The 2% undetected strains are primarily due to sample tracking errors, slow growth of the mutant strain during the process of pool construction, and mutations in the TagModule (data not shown). Taking into account only those strains that are detectable, our two pools contain 5,680 unique transposon mutants (2,420 mutant strains are in both pools) and represent transposon insertions in 3,345 unique protein-coding genes. For 1,675 genes, two or more independent mutants are contained in the pools (<xref ref-type="supplementary-material" rid="pgen.1002385.s001">Figure S1</xref>). For most pooled fitness experiments, we performed a single experiment on each of the upPool and dnPool.</p><p>Individual aliquots of each pool were frozen at &#8722;80&#176;C in glycerol (10% v/v). Prior to initiating a pool experiment, a single freezer aliquot of each pool was grown in LB aerobically at 30&#176;C to mid log phase (OD<sub>600</sub>&#8202;=&#8202;&#8764;2.0). At this point, we collected a sample (&#8764;1&#215;10<sup>9</sup> cells) that we term the &#8220;start&#8221;. The start sample represents the time 0 of the experiment and is the control experiment that we compare all of our growth conditions to. The same recovered cells were typically used to inoculate the &#8220;condition&#8221; media at a starting OD<sub>600</sub> of 0.01 or 0.02. For standard liquid media conditions, we typically collected condition samples after the cultures reached saturated growth, representing between 3 and 9 population doublings. Some conditions, such as LB, reach a high density and have more population doublings than certain minimal media conditions, such as butyrate as the sole source of carbon.</p><p>Liquid growth pool experiments were done in a number of formats. Aerobic minimal media experiments were done in 10 mL volumes with shaking at 200 rpm. Anaerobic experiments were conducted in hungate tubes with shaking at 200 rpm. Anaerobic experiments were set up in an anaerobic chamber (Coy) with a gas mix of 5% H<sub>2</sub>, 10% CO<sub>2</sub>, and 85% N<sub>2</sub>. Certain stress experiments in LB were performed in 1 mL volumes in the wells of a 24-well microplate. For these experiments, the microplate was grown in a Tecan Infinite F200 reader to measure the amount of growth inhibition caused by the stress. Our target stress concentration resulted in a &#8764;50% reduction of the growth rate.</p><p>For swimming motility experiments, we pipetted &#8764;1&#215;10<sup>8</sup> cells from the start culture into the matrix of an LB soft agar plate (0.25% w/v agar) and incubated the plate at 30&#176;C. After 1 or 2 days, we removed cells from the outer ring (i.e. the motile cells) using a sterile razor.</p><p>For heat shock survival experiments, we incubated the aliquots of the start cells in a 42&#176;C water bath for different amounts of time. After incubation, we used some of the cells for measuring viability by serial dilution and plating on LB. The remainder of the cells was used to inoculate a fresh tube of LB that was grown overnight (to avoid the possibility of detecting tags from dead cells). The tags from the overnight sample were hybridized and used as a measure of cell survival after heat shock. For cold survival experiments, we used the same method as for heat shock except we incubated the cells at 4&#176;C rather than 42&#176;C. We followed a similar method for stationary phase survival. However, in this instance, we left the pools in LB at 30&#176;C for days after the cultures reached saturation. Again, we used some of the cells for determining viability by plating on LB plates; additional cells were used to inoculate fresh LB media. The overnight growth of these fresh cultures was used for tag array hybridization and serve as a measure of mutant survival in stationary phase. Further details on the media used and the growth conditions for each of the 195 pool experiments are contained in <xref ref-type="supplementary-material" rid="pgen.1002385.s011">Table S3</xref>. Plots showing the survival of MR-1 cells after heat shock, cold incubation, and stationary phase are contained in <xref ref-type="supplementary-material" rid="pgen.1002385.s008">Figure S8</xref>.</p><p>Genomic DNA was isolated from each sample using either the DNeasy blood tissue kit with optional RNase treatment (Qiagen) or with a QIAxtractor genomic DNA robot (Qiagen). Approximately 100 ng of genomic DNA was used as a template to amplify the uptags from the upPool samples and the downtags from the dnPool samples using previously described primers and PCR conditions <xref ref-type="bibr" rid="pgen.1002385-Pierce1">[55]</xref>. We combined uptag and downtag PCR products (10 &#181;L of each) and hybridized to a single GenFlex 16K_v2 microarray (Affymetrix) that contains the tag complement sequences. Microarrays were hybridized, washed, labeled, and scanned as described <xref ref-type="bibr" rid="pgen.1002385-Pierce1">[55]</xref>.</p><p>Using Affymetrix .CEL files as a starting point, we first averaged the log<sub>2</sub> intensities across the 5 replicate probes for each tag to obtain values for each uptag and downtag. We computed the difference (the log ratio) between these values for the condition array and the start array, to give a fitness value for each strain. Sometimes we used an average of start arrays from other experiments instead of hybridizing a start array from that actual experiment; log-levels in these independent start experiments were highly correlated (r&gt;&#8202;=&#8202;0.95). We removed strains with the lowest 2% of levels in this average start array from the analysis. We normalized the fitness values for the strains so that the median fitness for each pool and for each chromosome (the main chromosome and the megaplasmid) was zero. After this normalization, some of our experiments showed significant effects based on which 96-well plate the strain had been grown in while we were preparing our pools. So, for all experiments, we also set the median fitness of each of these groups of 96 strains to zero. See <xref ref-type="supplementary-material" rid="pgen.1002385.s013">Table S5</xref> for all strain fitness data. We computed fitness values for each gene by averaging the fitness values for all of the insertions in that gene. If a gene had one or more &#8220;good&#8221; insertions (an insertion within the central 5&#8211;80% portion of the gene), then we used only those good insertions to compute the average. See <xref ref-type="supplementary-material" rid="pgen.1002385.s014">Table S6</xref> for all gene fitness data.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To estimate the reliability of each fitness value, we took advantage of our 14 control experiments (measurements of the start pools after independent recoveries from the freezer) and the fact that we have more than one fitness measurement for most genes (i.e., more than one strain, or the single strain for the gene is in both pools). We used an approach similar to that of Efron <italic>et al</italic>. <xref ref-type="bibr" rid="pgen.1002385-Efron1">[76]</xref>. We first computed a t-like test statistic, which was:<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1002385.e001.jpg" /></disp-formula>
<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1002385.e002.jpg" /></disp-formula>where x are the measurement(s) for the gene, &#956; is their average, n is the number of measurements, and &#936;&#8202;=&#8202;median(STD(x)), that is, the median across all genes with more than one measurement of the standard deviation of that gene's measurements. We transformed our test statistic into Z scores that follow a normal distribution in the absence of biological signal by using the control experiments; we transformed the distribution separately for genes with 1, 2, 3, or &gt;&#8202;=&#8202;4 measurements. We used conditions that we had repeated to verify that these Z scores were appropriate (i.e., the rate of discordant outliers with high |Z| values was about the same as expected by chance; data not shown). See <xref ref-type="supplementary-material" rid="pgen.1002385.s015">Table S7</xref> for Z score data.</p><p>Aerobic single strain growth assays were performed in 96-well microplates in a Tecan Sunrise plate reader at 30&#176;C with readings every 15 minutes. Anaerobic single strain growth assays were performed in a DTX880 plate reader (Beckman) housed in an anaerobic chamber (Coy) at 30&#176;C with readings every 30 minutes. All microplate growth assays contained 150 &#181;L per well at a starting OD<sub>600</sub> of 0.02. Before all single strain growth assays, we isolated a single colony of the mutant strain and confirmed the expected location of the transposon insertion by PCR with a transposon specific primer and a genome primer. We calculated doubling times for growth curves using a logistic algorithm implemented in R. To calculate the relative growth rate for transposon mutants in <xref ref-type="fig" rid="pgen-1002385-g001">Figure 1C</xref>, we divided the doubling time of the transposon mutant by the doubling time of wild-type MR-1. All single strain growth assays were performed a minimum of three times.</p><p>We measured gene expression in wild-type MR-1 and in single transposon mutant strains. Cells were harvested after RNAprotect treatment (Qiagen) in either early exponential growth or one hour after transfer to an experimental media. For the transfer experiments, all cultures were initially cultured in DL-lactate minimal media to early exponential phase prior to transfer to the experimental media. For all experiments, we collected &#8764;2&#215;10<sup>9</sup> cells, isolated total RNA with a RNeasy mini kit (Qiagen), and synthesized Alexa Fluor 555 labeled cDNA with the SuperScript Plus Indirect cDNA Labeling Module (Invitrogen).</p><p>Labeled cDNA was hybridized to custom Nimblegen oligonucleotide microarrays according to the manufacturer's instructions. We used two gene expression microarray designs, a 4-plex microarray that allows hybridizing 4 samples to different regions of a single slide, and also a 12-plex microarray. The 4-plex microarray had 66,228 probes, which reduced to 61,178 after removing potential cross-hybridizing probes with BLAT, or roughly 15 probes per gene. The 12-plex microarray had 42,598 probes, which reduced to 40,881 after removal of cross-hybridizing probes. After removing potential cross-hybridizing probes, we made the distribution of the condition match that of the control (i.e., using quantile normalization), used local regression (lowess) to eliminate any bias in the log<sub>2</sub> ratio by probe intensity, and set the median normalized log<sub>2</sub> levels of the probes to zero (separately for each scaffold). The final log<sub>2</sub> ratio for each gene was the average of these normalized values for its probes; we removed values for genes with less than 4 measurements. These data are available on MicrobesOnline. To verify the quality of each experiment, we checked the correlation of log ratios between adjacent genes that are predicted to be in the same operon and the average absolute difference within these pairs. Most comparisons had operon correlations above 0.8; some experiments had lower correlations but also had low average differences, indicating that the correlation was low because there was less biological signal to detect.</p><p>To complement the mutant phenotypes of single transposon mutant strains, we introduced an intact copy of the mutated gene on the broad-range plasmid pBBR1MCS5, which contains a gentamicin resistance marker <xref ref-type="bibr" rid="pgen.1002385-Kovach1">[77]</xref>. If possible, we used the native promoter of the gene to drive expression. Otherwise, we relied on the activity of the pBBR1MCS5 <italic>lac</italic> promoter to express the complementation gene. The complementation plasmids were constructed with circular polymerase extension cloning (CPEC) <xref ref-type="bibr" rid="pgen.1002385-Quan1">[78]</xref> using primers <named-content content-type="gene">5&#8242;-<underline>GCTCTAGAACTAGTGGATCCCCC</underline>
</named-content>(N) and <named-content content-type="gene">5&#8242;- <underline>GATATCGAATTCCTGCAGCCC</underline>
</named-content>(N) where (N) represents genome-specific primer sequences used to amplify the gene from genomic DNA. The underlined regions represent sequences homologous to the pBBR1MCS5 backbone. To prepare the vector for CPEC, we amplified pBBR1MCS5 with primers <named-content content-type="gene">5&#8242;-GGGCTGCAGGAATTCGATATC</named-content> and <named-content content-type="gene">5&#8242;- GGGGGATCCACTAGTTCTAGAGC</named-content>. Following amplification, the template vector was digested with DpnI. All PCR and CPEC reactions were performed with Phusion high fidelity DNA polymerase (New England Biolabs). Both the vector and insert were gel purified with the Zymoclean Gel Recovery Kit (Zymo Research). The CPEC reaction consisted of 50 ng of linearized pBBR1MCS5, 25 ng of the complementation gene PCR product, and was cycled with the following protocol: initial denaturation for 15 seconds at 98&#176;C, 4 cycles of 98&#176;C for 30 seconds, 55&#176;C for 30 seconds, and 72&#176;C for 3.5 minutes, and a final extension at 72&#176;C for 3 minutes. The CPEC reaction was transformed into chemically competent TOP10 cells (Invitrogen) and plated on LB with gentamicin. Complementation constructs were sequence-verified with primers comp_for_seq, comp_rev_seq, and in some instances, additional internal gene primers (see <xref ref-type="supplementary-material" rid="pgen.1002385.s016">Table S8</xref> for full list of primer sequences used in this study). Complementation plasmids were transformed into <italic>E. coli</italic> donor strain WM3064 and delivered into MR-1 via conjugation. Single MR-1 colonies carrying the complementation plasmids were selected on LB plates with gentamicin and assayed for growth in a microplate format as described above. See <xref ref-type="supplementary-material" rid="pgen.1002385.s017">Table S9</xref> for full list of complementation plasmids used in this study.</p><p>TIGR roles and subroles are associated with some gene families in the TIGRFam database <xref ref-type="bibr" rid="pgen.1002385-Peterson1">[29]</xref>. We obtained TIGRFam assignments from MicrobesOnline along with their associated roles and subroles. To exclude functional assignments that were not specific, we removed assignments without a subrole or that matched the strings &#8220;unknown&#8221; or &#8220;other&#8221;. This left us with 978 assignments. To predict the functional classification of genes from fitness and/or expression data, we used a standard implementation of random forests (randomForest 4.5&#8211;36 in R-2.11) and default settings. A random forest is a collection of decision trees, each of which makes their own predictions. The random forest's prediction is that predicted by the largest number of decision trees, and the confidence of the prediction is the proportion of trees in the forest that make that prediction. Before applying the random forest, we subtracted the mean from each experiment and replaced missing values with zeroes. To assess the predictions without being biased by the training data, we used 10-fold cross-validation: we trained the classifier on 90% of the genes with known subroles and then made predictions for the remaining 10% of the genes. We repeated this 10 times so that we had predictions for every gene. Because genes in the same operon tend to be functionally related, we ensured that genes in the training set (the 90%) were not in the same operons as the genes we made predictions for.</p>