<p xmlns:ns0="http://www.w3.org/1999/xlink">The Mexican individuals analyzed in this project come from two sources: a panel of 492 Mexican parent-offspring trios recruited from Mexico City as part of a previous genome-wide association study (MEX1) <xref ref-type="bibr" rid="pgen.1002410-Hancock1">[14]</xref>, and 23 HapMap Phase3 Mexican trios recruited from Los Angeles, California (MEX2; <ext-link ext-link-type="uri" ns0:href="http://hapmap.org">http://hapmap.org</ext-link>). For estimating locus-specific ancestry, we used the HapMap CEU (N&#8202;=&#8202;88) and YRI (N&#8202;=&#8202;100) individuals for the ancestral populations. To analyze the European component of the admixed genome, we augmented the Mexican datasets with individuals recruited from Dublin, Ireland (N&#8202;=&#8202;43), Rome, Italy (45), Warsaw, Poland (N&#8202;=&#8202;45) and Porto, Portugal (N&#8202;=&#8202;43). For the Indigenous American component analyses, we combined the data generated in two previous studies <xref ref-type="bibr" rid="pgen.1002410-Li1">[2]</xref>, <xref ref-type="bibr" rid="pgen.1002410-Bigham1">[16]</xref>. Four Mayan individuals with substantial European admixture are removed. The combined set used for the subsequent analyses includes 14 individuals from Guerrero, Mexico (two Nahua, seven Mixtec and five Tlapanec), 24 Mayan individuals from the Yucatan Peninsula, 24 Quechua collected in Cerro de Pasco, Peru, 25 individuals of largely Aymara ancestry collected in La Paz, Bolivia, 13 Karitiana and eight Surui from Brazil, seven Colombians, and 14 Pima. Because the sample sizes for Nahua, Mixtec and Tlapnec are small, and all individuals were recruited from the same state, we considered these individuals as one group. <xref ref-type="supplementary-material" rid="pgen.1002410.s004">Table S1</xref> summarizes the individuals used for each analysis.</p><p>Genotyping and quality control procedures have been described in the primary publications for each dataset, except for the dataset of 176 European individuals. Briefly, MEX1 and the HGDP individuals were genotyped on Illumina 550K and on 650K Beadchip, respectively. The Indigenous American individuals from Bigham et al. (2009) were genotyped on Affymetrix 1M SNP arrays <xref ref-type="bibr" rid="pgen.1002410-Bigham1">[16]</xref>. The set of 176 European individuals were genotyped using Illumina HumanHap300 arrays; this dataset originally included 180 individuals; four individuals were found with non-negligible non-European ancestry and were excluded. SNPs with a call rate of less than 95% were excluded. The number of individuals and markers used for each analysis is summarized in <xref ref-type="supplementary-material" rid="pgen.1002410.s004">Table S1</xref>. We used BEAGLE to construct haplotypes for Mexican trios <xref ref-type="bibr" rid="pgen.1002410-Browning1">[34]</xref>. As children provide no additional information regarding population structure or adaptation, they are not used in subsequent analyses.</p><p>Continental-level admixture proportions were estimated two ways: (1) a model-based clustering algorithm implemented in <italic>frappe</italic>
<xref ref-type="bibr" rid="pgen.1002410-Tang3">[35]</xref>, and (2) average locus-specific ancestries across all markers. Locus-specific ancestry was estimated with SABER+, an extension of a previously described approach, SABER, that uses a Markov-Hidden Markov Model <xref ref-type="bibr" rid="pgen.1002410-Tang1">[12]</xref>. SABER+ differs from SABER in implementation of a new algorithm, an Autoregressive Hidden Markov Model (ARHMM), in which haplotype structure within the ancestral populations is adaptively constructed using a binary decision tree based on as many as 15 markers, and which therefore does not require a priori knowledge of genome-wide ancestry proportions (Johnson et al., in preparation). In simulation studies, the ARHMM achieves accuracy comparable to HapMix <xref ref-type="bibr" rid="pgen.1002410-Price2">[36]</xref> but is more flexible in modeling the three-way admixture in the Mexican population and does not require information about the recombination rate.</p><p>HapMap CEU and YRI individuals were used as the reference ancestral populations. Based on <italic>frappe</italic> and supported by PCA, 50 individuals in MEX1 set have more than 95% Indigenous American ancestry. These individuals were initially used to approximate the Indigenous American ancestors in the locus-specific ancestry analyses; an iterative procedure is used to identify and correct for the non-Indigenous American segments in these individuals. Accuracy of the locus-specific ancestry is verified by performing a PC analysis, treating each individual as three non-admixed genomes, Mex<sup>EUR</sup>, Mex<sup>AMR</sup>, and Mex<sup>AFR</sup> (see section &#8220;subspace PCA&#8221; below).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We implemented this algorithm to accommodate the large amount of missing genotype data in partially masked virtual genomes, and used it to derive all the PCA results reported here. The statistical theory of the algorithm in a general data mining context can be found in <xref ref-type="bibr" rid="pgen.1002410-Raiko1">[13]</xref>; however, various modifications are required for the current setting, as described below. Let <bold><italic>G<sup>h</sup></italic></bold> (h&#8202;=&#8202;1,2) be two <italic>N</italic>&#215;<italic>M</italic> matrices, in which <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002410.e002.jpg" /></inline-formula> denote the unordered pair of alleles at SNP <italic>m</italic> (<italic>m</italic>&#8202;=&#8202;<italic>1</italic>,&#8230;,<italic>M</italic>) in individual <italic>n</italic> (<italic>n</italic>&#8202;=&#8202;<italic>1</italic>,&#8230;,<italic>N</italic>); the columns of <bold><italic>G<sup>h</sup></italic></bold> are standardized to have mean 0 and variance 1. To compute the subspace spanned by the first <italic>k</italic> principal components (PC), we begin by finding a matrix decomposition, <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002410.e003.jpg" /></inline-formula>, which minimizes the reconstruction error, <italic>R</italic>, defined as:<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1002410.e004.jpg" /></disp-formula>subject to the constraints that the column vectors of <bold>A</bold> are of unit norm and mutually orthogonal and the row vectors of <bold>S</bold> are also mutually orthogonal. Here <bold>A</bold> is a <italic>N</italic>&#215;<italic>d</italic> matrix, <bold><italic>S</italic></bold> is a <italic>M</italic>&#215;<italic>d</italic> matrix, and <italic>d&lt;N&#8804;M</italic> represents the desired number of leading PC's. The algorithm we use is a generalized instance of the coordinate descent approach <xref ref-type="bibr" rid="pgen.1002410-Friedman1">[37]</xref>, which iteratively optimizes matrix <bold><italic>A</italic></bold> for fixed <bold><italic>S</italic></bold> and then optimizes <bold><italic>S</italic></bold> fixing <bold><italic>A</italic></bold> according to the rules:<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1002410.e005.jpg" /></disp-formula>
<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1002410.e006.jpg" /></disp-formula>where <italic>&#955;</italic> is a learning rate, the superscripts, <italic>r</italic>, indicate iteration, and the subscripts, <italic>j</italic>, denote the <italic>j</italic>-th column of a matrix. It can be shown that the columns of <bold><italic>A</italic></bold> and <bold><italic>S</italic></bold> span the subspace of the first <italic>d</italic> PCs, and that the leading PCs can be computed by orthogonalizing the columns of <bold><italic>A</italic></bold> and <bold><italic>S</italic></bold>
<xref ref-type="bibr" rid="pgen.1002410-Raiko1">[13]</xref>. To evaluate the accuracy of our modified ssPCA approach, we applied it in parallel with Eigenstrat <xref ref-type="bibr" rid="pgen.1002410-Price1">[21]</xref> to the intact Mexican genomes, and found the leading PCs produced by the two algorithms were virtually identical, up to a permutation of signs.</p><p>We carried out two simulation experiments to evaluate the impact of statistical uncertainties associated with estimating locus-specific ancestry, and to investigate the performance of the ssPCA approach.</p><p>In the first set of simulations, we created 10 datasets in which 400 admixed genomes were modeled to mimic a Latino population: each individual draws chromosomal segments from European and Indigenous American ancestry, and the proportion of Indigenous American ancestry in each individual matches what we observed in MEX1. For 200 individuals, European-derived segments were sampled from the HapMap CEU haplotypes, representing Northern and Western European ancestry, while for the remaining 200 individuals, the European-derived segments were sampled from Mex<sup>EUR</sup> inferred from the actual Mexican genotype data, representing Southern European ancestry. The chromosomal segments from CEU and Mex<sup>EUR</sup> in the admixed individuals were treated as the true European virtual genomes.</p><p>To evaluate the potential impact of statistical uncertainty, we introduced random errors in which the true identities of European vs. Indigenous American segments were switched with probability <italic>&#949;</italic>. The top PC for each set of simulated virtual genomes (at each of 8 error rates, <italic>&#949;</italic>&#8202;=&#8202;0.01&#8211;0.20) was computed with ssPCA. We evaluate the effect of these errors by calculating a confusion fraction, <italic>&#958;</italic>, that quantifies the accuracy with which the estimated first PC separates individuals with Northern vs. Southern European ancestry, and is defined as the proportion of individuals that lie on the &#8220;wrong&#8221; side of a threshold that best separates the two groups. Thus, <italic>&#958;</italic> can range from 0 (perfect separation) to nearly 50% (complete confusion as would be observed for genetically homogenous groups). Finally, we analyze each of the 10 datasets using SABER+, exactly as was done for real data: apply ssPCA to estimate substructure, and calculate a confusion fraction. The results for this simulation experiment are depicted in <xref ref-type="supplementary-material" rid="pgen.1002410.s002">Figure S2</xref>, and show that the confusion fraction increases substantially, from a mean of 2.14% to 17.5%, at error rates between 0.03 and 0.05. Using SABER+ on these same 10 datasets yields a mean confusion fraction of 1.58%, which corresponds to an error rate &lt;0.02 (indicated by the arrow in <xref ref-type="supplementary-material" rid="pgen.1002410.s002">Figure S2</xref>).</p><p>In a second set of simulations to investigate the ability of the ssPCA approach to deal with missing data, we created five datasets in which the proportion of genome-wide European ancestry in each of 400 admixed genomes was fixed at either 50% or 30%, respectively. Applying SABER+ and ssPCA to these datasets yields mean confusion fractions of 0 and 0.7%, respectively, indicating that our approach performs well for situations such as the one described here, where mean genome-wide continental ancestry proportions are above 30% for both the European and the Indigenous American components.</p><p>We used the number of ancestry blocks in an individual as summary statistics. Tracing through a pedigree of <italic>T</italic> generations, the expected number of recombination events in a haploid genome is 0.01&#215;<italic>TL</italic>, where <italic>L</italic> is the total genome length (taken to be 3435cM <xref ref-type="bibr" rid="pgen.1002410-Kong1">[38]</xref>). Under a hybrid-isolation model and assuming a genome-wide ancestry proportion of z, a fraction of <italic>2&#215;z(1&#8722;z)</italic> of the recombination events occurs between two haplotypes of opposite ancestry and thus leads to transitions in ancestry. When we count the number of ancestry blocks in the real data, we do not observe recombination events that occur between two haplotypes of the same ancestry. Hence the expected number of ancestry switches in a diploid genome is <italic>B&#8202;=&#8202;(2&#215;2&#215;0.01)&#215;TL&#215;z(1&#8722;z)</italic>, and each ancestry switch creates one additional ancestry block. When there is no ancestry switch in a genome, the number of ancestry blocks is defined to be the same as the number of chromosomes. Therefore, for each specific time of admixing, <italic>T</italic>, we computed the expected number of ancestry blocks as <italic>B+2&#215;22</italic>, with the genome-wide ancestry proportion, <italic>z</italic>, varying from 0 to 1 at 100 equally spaced grid points. Each curve in <xref ref-type="fig" rid="pgen-1002410-g002">Figure 2D</xref> shows the expected number of ancestry blocks as a function of admixture proportions for a specific admixing time. The estimated numbers of European and non-European ancestry blocks from the Mexican individuals were tallied and compared to the expected values. To assess the impact of uncertainty associated with estimating the number of ancestry blocks, we note that errors in estimating locus-specific ancestry often create very short ancestry blocks. Hence, we simulated admixed genomes according to the hybrid-isolation model, but removed extremely short blocks (segments with &lt;10 SNPs) from both simulated genomes and real data. The estimated admixing time remained the same under this alternative analysis, suggesting the estimated admixing time is relatively robust. The hybrid-isolation model was chosen because of the mathematical simplicity; under a more realistic continuous gene-flow model, the estimated times of admixing should be interpreted as an approximation of average admixing time, weighted by the relative level of gene-flow in each generation.</p><p>To assess the sub-continental population structure, each Mexican genome was partitioned into three non-admixed genomes, by masking (i.e. setting to missing) alleles from all but one ancestral population. In other words, the European component of a Mexican's genome (Mex<sup>EUR</sup>) was derived by treating as missing all alleles whose origins were inferred as African or Indigenous American. For within European analysis, we applied ssPCA on the dataset consisting of Mex<sup>EUR</sup> (including both Mexico City and HapMap samples), 88 HapMap CEU and 176 European individuals from four cities: Dublin (Ireland), Warsaw (Poland), Rome (Italy) and Porto (Portugal). Because of the limited number of informative haplotype segments, Mexican individuals with less than 25% European ancestry were excluded from this analysis. In an analogous fashion, we analyzed the Mexican component of the genome, Mex<sup>AMR</sup>, along with 129 indigenous Indigenous American individuals representing 8 populations (<xref ref-type="supplementary-material" rid="pgen.1002410.s004">Table S1</xref>). Because the African ancestry is low in both Mexican cohorts (3% and 5%, respectively), we did not analyze the within-Africa population structure.</p><p>or all SNPs with frequencies between .05 and .95 in the respective populations, iHS was calculated following Voight et al. <xref ref-type="bibr" rid="pgen.1002410-Voight1">[8]</xref> with two modifications. First, haplotype homozygosity scores for a core SNP is computed on the subset of haplotypes in which the core SNPs are derived from a specific population. If a haplotype is truncated because of an ancestry change, the haplotype beyond the ancestry switch point is considered different from all other haplotypes in the corresponding interval. We have also considered an alternative strategy in which only haplotypes that do not have an ancestry change within 400 SNPs from the core SNPs are included in the calculation; the results are virtually identical. Second, instead of binning SNPs by the inferred ancestral allele frequencies and calculating the standard deviation of iHS in each bin, we used a quantile regression to estimate the 25th- and 75th-percentile of the empirical null distribution as a function of the minor allele frequency. The raw iHS scores were then normalized by the estimated inter-quartile range within each chromosome; the resulting standardized iHS scores fit a standard normal distribution well. To define regions that may harbor recently adaptive alleles, we seeded a region by a window of 50kb around a SNP with extreme iHS scores; we then successively scanned to the left and to the right, 50kb a time, merging neighboring regions in which at least one SNPs has an |iHS|&gt;2.5 (which represents the 99th-percentile of the scores); finally, the top 10 list in <xref ref-type="table" rid="pgen-1002410-t001">Table 1</xref> requires that at least 10% of the SNPs in the region have |iHS|&gt;2.5. The proportion of SNPs with high |iHS| was the criterion used by Pickrell et al. <xref ref-type="bibr" rid="pgen.1002410-Pickrell1">[9]</xref>. It has been suggested that genome-wide and locus-specific ancestries may show particular poor correlation at loci under selection compared to neutrally evolving loci <xref ref-type="bibr" rid="pgen.1002410-Qin1">[39]</xref>. We did not observe such a trend at loci with the highest iHS scores in either MEX<sup>AMR</sup> or MEX<sup>EUR</sup>.</p>