<p>Note that the following section only provides a brief summary of the methods applied to our study. A much more detailed description can be found in <xref ref-type="supplementary-material" rid="pgen.1002548.s007">Text S1</xref>.</p><p>For inclusion in PDGene, a study has to meet three criteria: 1) It must evaluate the association between a bi-allelic genetic polymorphism (minor allele frequency &#8805;0.01 in the healthy control population of at least one study) and Parkinson's disease (PD) risk in datasets comprised of both affected (defined as clinically and/or neuropathologically diagnosed &#8220;Parkinson's disease&#8221;) and unaffected individuals; 2) it must be published in a peer-reviewed journal; 3) it must be published in English. For this manuscript, we also included data on ten SNPs generated in the GEO-PD Consortium datasets <xref ref-type="bibr" rid="pgen.1002548-Satake1">[14]</xref>, <xref ref-type="bibr" rid="pgen.1002548-Sharma1">[23]</xref> and obtained data for the newly identified SNP rs7077361 in <italic>ITGA8</italic> from the Japanese GWAS dataset <xref ref-type="bibr" rid="pgen.1002548-Satake1">[14]</xref>.</p><p>In brief, genetic association data of the following studies were excluded from the meta-analyses (see <xref ref-type="supplementary-material" rid="pgen.1002548.s007">Text S1</xref> for details): family-based studies without available subject-level data (however, unrelated case-control data enriched for familial cases were not excluded), studies investigating only disease controls, multi-allelic polymorphisms, and studies of polymorphisms in mitochondrial DNA. We also excluded genetic data of apparently &#8220;poor&#8221; quality if discrepancies could not be resolved after contacting the study authors (e.g. inadequate genotyping/sequencing protocols or discrepancies in terms of allele names or frequencies when compared with public databases; more details can be found in <xref ref-type="supplementary-material" rid="pgen.1002548.s007">Text S1</xref>).</p><p>Our literature searches until March 31<sup>st</sup>, 2011, yielded 27,210 articles, which were screened for eligibility using the title, abstract, or full-papers, as necessary. Additional screening of bibliographies in reviews, published meta-analyses, and original genetic association studies were also performed. Overall, full text versions of 1,534 articles were obtained. Following the inclusion and exclusion criteria outlined above, 828 articles were included in PDGene until March 31<sup>st</sup> 2011 (also see <xref ref-type="fig" rid="pgen-1002548-g003">Figure 3</xref>).</p><p>Random-effects allelic meta-analyses <xref ref-type="bibr" rid="pgen.1002548-DerSimonian1">[31]</xref> were performed if a minimum of four independent datasets existed per polymorphism. Summary odds ratios [ORs] and 95% confidence intervals [CIs] were calculated irrespective of ethnic descent as well as for distinct ethnic groups (i.e. Caucasians, and Asians) if sufficient data were available. In addition, we performed a number of sensitivity analyses (excluding the initial studies and datasets in which HWE was violated in control individuals), systematically assessed between-study heterogeneity (via <italic>I</italic>
<sup>2</sup>), and assessed the credibility of each at least nominally significant meta-analysis result by calculating Bayes factors (BF; here expressed as log10(BF)="logBF&#8221;) <xref ref-type="bibr" rid="pgen.1002548-Ioannidis1">[25]</xref> and by determining a grading score developed by the Human Genome Epidemiology Network (HuGENet) <xref ref-type="bibr" rid="pgen.1002548-Ioannidis2">[26]</xref>, <xref ref-type="bibr" rid="pgen.1002548-Khoury1">[27]</xref>.</p><p>This is of particular importance in meta-analyses of published association data and was carefully addressed here: First, we added <italic>publicly</italic> available GWAS data <xref ref-type="bibr" rid="pgen.1002548-Maraganore2">[10]</xref>, <xref ref-type="bibr" rid="pgen.1002548-Pankratz1">[12]</xref>, <xref ref-type="bibr" rid="pgen.1002548-SimnSnchez1">[13]</xref> to the vast majority of SNPs. Since these data are typically unbiased, this should decrease the potential for small-study bias/publication bias. Secondly, for 147 SNPs of the core PDGene meta-analyses that showed statistically suggestive results (<italic>P</italic>&#8804;0.1), we obtained additional data from all currently published, but <italic>not publicly</italic> available GWAS datasets, further decreasing a potential impact of small-study bias/publication bias. Thirdly, we directly assessed the evidence for small study bias by applying a recently proposed regression test <xref ref-type="bibr" rid="pgen.1002548-Harbord1">[32]</xref> on all nominally significant (<italic>P</italic>&lt;0.05) meta-analysis results. The results of these analyses are fully displayed in <xref ref-type="supplementary-material" rid="pgen.1002548.s005">Table S1</xref>.</p><p>We obtained individual-level genotype data for all publicly available PD GWAS datasets from NCBI's &#8220;dbGAP&#8221; database (a total of three <xref ref-type="bibr" rid="pgen.1002548-Maraganore2">[10]</xref>, <xref ref-type="bibr" rid="pgen.1002548-Pankratz1">[12]</xref>, <xref ref-type="bibr" rid="pgen.1002548-SimnSnchez1">[13]</xref> at the time of the datafreeze, March 31<sup>st</sup>, 2011). Genotype data were cleaned using standard procedures, followed by imputation of untested genotypes (using reference panels from HapMap and the 1000 Genomes Project), and association analyses incorporating imputation uncertainty (case-control datasets only), age, sex, and population stratification. Overall, this procedure led to a total of 7,723,931 unique SNPs, 7,123,920 of which were present in at least two, and 711,271 in at least three datasets. Meta-analyses (either combining test-statistics and standard errors using random-effects models, or by combining P-values weighted by sample size, see <xref ref-type="supplementary-material" rid="pgen.1002548.s007">Text S1</xref> for more details) were performed on the 7,123,920 SNPs present in at least two of the GWAS datasets.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">After completion of all data-management and analysis steps, all study-specific variables, genotype data (except for GWAS), and meta-analysis plots are posted on a dedicated, publicly available, online adaptation of the PDGene database using the same software and code as our databases for Alzheimer's disease <xref ref-type="bibr" rid="pgen.1002548-Bertram1">[33]</xref> and schizophrenia <xref ref-type="bibr" rid="pgen.1002548-Allen1">[34]</xref>. The online database is hosted by the &#8220;Alzheimer Research Forum&#8221; and can be accessed via its own designated URL (<ext-link ext-link-type="uri" ns0:href="http://www.pdgene.org">http://www.pdgene.org</ext-link>).</p><p>The database software can easily be ported to other genetically complex diseases and will be made available on a collaborative basis to interested researchers upon request.</p>