<p>Prior to their implementation, all animal procedures were reviewed and approved by the care and use committees at the United States Department of Agriculture (USDA), Agricultural Research Service (ARS) Meat Animal Research Center (USMARC) in Nebraska, the USDA, ARS, Sheep Experiment Station (USSES) in Idaho, and Washington State University in cooperation with the USDA, ARS, Animal Disease Research Unit (ADRU).</p><p>The USMARC (Nebraska) sheep population was sampled in 2003 (n&#8202;=&#8202;3545) and used to select 69 matched case-control pairs of 5- to 9-year-old ewes for the GWAS. The same population was also used to select 61 matched case-control pairs of 4-year-old ewes for analyzing <italic>TMEM154</italic> haplotypes as risk factors for OPP infection. Animals not used in matched case-controls were used in unmatched cohort studies for validation as shown in <xref ref-type="supplementary-material" rid="pgen.1002467.s002">Table S2</xref>. Animals were not members of more than one group. The USMARC sheep population is a relatively diverse flock with more than ten breeds representing genetic diversity for traits such as fertility, prolificacy, maternal ability, growth rate, carcass leanness, wool quality, mature weight and longevity <xref ref-type="bibr" rid="pgen.1002467-Heaton1">[47]</xref>.</p><p>The USMARC population was sampled again in 2010 and used to select a cohort of 280 ewes, 4- to 5-year-old, and raised in similar conditions as those sampled in 2003. The purpose was to determine if the association of <italic>TMEM154</italic> haplotypes with OPP infection was reproducible in animals sampled seven years later.</p><p>The USSES (Idaho) sheep population was sampled in 2004 and 2008 and used to select cohorts of 309 and 365 mature ewes, respectively. The purpose was to determine if an association of <italic>TMEM154</italic> haplotypes with OPP infection was evident in another research flock that was geographically and historically distinct from the Nebraska flock. The USSES sheep population contains Columbia, Rambouillet, and Polypay breeds.</p><p>The private Polypay sheep flock (Iowa) was sampled in 2009 and used to select a cohort of 218 mature ewes. The purpose was to determine if an association of <italic>TMEM154</italic> haplotypes with OPP infection was evident in a commercial flock distinct from those in Nebraska and Idaho. This commercial flock was chosen based on its availability.</p><p>Whole blood samples for serum fractionation and DNA extraction were drawn from the jugular vein into S-Monovette serum Z and EDTA KE 9 ml syringes, respectively (Sarstedt, Newton, NC, USA). Laboratory diagnosis for OPP was performed at the Washington Animal Disease Diagnostic Laboratory (Pullman, WA, USA) with a Caprine Arthritis Encephaltitis Virus (CAEV) competitive-inhibition ELISA (cELISA). This CAEV cELISA is applicable for the detection of OPPV antibodies in sheep <xref ref-type="bibr" rid="pgen.1002467-Herrmann1">[45]</xref>, <xref ref-type="bibr" rid="pgen.1002467-Herrmann2">[48]</xref>. Briefly, this assay uses a proprietary monoclonal antibody derived from the fusion of goat splenocytes and mouse myeloma cells (VMRD, Inc., Pullman, WA, USA). This antibody is conjugated to horeseradish peroxide and is used to compete with serum antibodies for the CAEV antigen bound to the microtiter plate. Additional testing for OPP was performed at USMARC and ADRU with CAEV cELISA kits, according to manufacturer's instruction (VMRD, Inc., Pullman, WA, USA).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Sixty-nine pairs of ewes were selected from a total of 736 in the 5- to 9-year-old age class. The OPPV seroprevalence of the 736 ewes was 43%. In dominant and co-dominant models, our GWAS design had a detectable RR of genetic association that ranged from two to six with 69 paired case-controls, 50,000 SNPs, a false-positive rate (alpha) of 0.05, and a false-negative rate (beta) of 0.1 (simulation data not shown <xref ref-type="bibr" rid="pgen.1002467-Menashe1">[49]</xref>). In a co-dominant model of inheritance with a disease prevalence of 0.43, the minimum detectable RR was less than 2 for marker allele frequencies between 0.15 and 0.50, and LD values between 0.7 to 1.0. In a dominant model of inheritance, the minimum detectable RR ranged from 2 to 6 for conditions similar to those above. There were not enough matched pairs in this design to detect GWAS of recessively inherited disease risk alleles. SNP genotypes for the OvineSNP50 BeadChip DNA samples were measured and scored at GeneSeek Inc. (Lincoln, NE, USA), according to manufacturer's instructions (Illumina, Inc., San Diego, CA, USA). For determining the number of SNPs that performed reliably with the set of 138 ovine samples, a GenCall score greater than 0.7 was used as a cutoff and was determined by clustering and genotype calling algorithms provided by the manufacturer (Illumina, Inc., San Diego, CA USA). All single SNPs were analyzed for association with infection using PLINK v1.07 software <xref ref-type="bibr" rid="pgen.1002467-Purcell1">[50]</xref>, as described: <ext-link ext-link-type="uri" ns0:href="http://pngu.mgh.harvard.edu/purcell/plink/">http://pngu.mgh.harvard.edu/purcell/plink/</ext-link>.</p><p>The combined relative risk was assessed in sheep cohorts using the glimmix procedure of SAS 9.2 (SAS Institute, Cary, NC). The serological status of OPP was fit as the dependent variable in a Poisson model with a log link to give unbiased estimates of relative risk and slightly conservative (broad) confidence intervals <xref ref-type="bibr" rid="pgen.1002467-McNutt1">[51]</xref>. Breed and risk/nonrisk diplotype status were treated as independent fixed variables, where the risk was defined as the presence of at least one <italic>TMEM154</italic> haplotype allele 2 or 3. Age was included as a covariate and the cohort was treated as a random variable.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Ovine BACs predicted to contain <italic>TMEM154</italic> were identified from those mapped to the ovine draft genome sequence <ext-link ext-link-type="uri" ns0:href="http://www.livestockgenomics.csiro.au/sheep/oar2.0.php">http://www.livestockgenomics.csiro.au/sheep/oar2.0.php</ext-link>). BACs were isolated from an arrayed 10&#8211;12&#215; sheep BAC library (CHORI-243, <xref ref-type="bibr" rid="pgen.1002467-Dalrymple1">[52]</xref>), cultured, and the BAC DNA was purified. The BACs were derived from the Texel ram used for the ovine genome sequencing project (USMARC animal no. 200118011). Pooled samples of the four BACs (CH243-492L14, 270 kb; CH243-229A18, 147 kb; CH243-363J1, 329 kb; and CH243-426G18, 279 kb) were sequenced by synthesis with conditions optimized for 600 bp read lengths, according to manufacturer's instructions (Roche Applied Sciences, Branford, CT, USA). DNA sequences were assembled <italic>de novo</italic> with Newbler software provided by the manufacturer and the contigs were evaluated and viewed with Consed <xref ref-type="bibr" rid="pgen.1002467-Gordon1">[53]</xref>. Contig assembly made use of information and sequence available for cattle and sheep at the National Center for Biotechnology Information (NCBI) and International Sheep Genomics Consortium (ISGC), respectively. A 78 kb region of genomic DNA sequence containing the complete predicted <italic>TMEM154</italic> gene was assembled with 70 k reads and 25 Mb of sequence. Four large contigs were manually joined with information derived from ovine mRNA sequences, and the annotated 78 kb sequence was deposited in GenBank (accession number HM355886).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Ovine <italic>TMEM154</italic> exons were genotyped by Sanger sequencing of PCR fragments amplified from genomic DNA (<xref ref-type="supplementary-material" rid="pgen.1002467.s004">Table S4</xref>). DNA extraction and genetic analyses were performed in a manner similarly to that previously described <xref ref-type="bibr" rid="pgen.1002467-Heaton1">[47]</xref>. Briefly, a 1,000 bp PCR product containing each exon was sequenced in the 138 matched case-control sheep and 96 rams from a diverse panel of common U.S. sheep breeds (MARC Sheep Diversity Panel version 2.4) <xref ref-type="bibr" rid="pgen.1002467-Heaton1">[47]</xref>. After scoring polymorphisms from these 234 sheep in all exons, a second round of nested PCR fragments were designed so that: 1) a 700 bp amplicon was fully nested within each previous 1,000 bp amplicon, and 2) the amplification primers for the 700 bp products did not bind to polymorphic sites discovered from sequencing the 1,000 bp on the genome (<xref ref-type="supplementary-material" rid="pgen.1002467.s004">Table S4</xref>). The combined Sanger sequences from each animal were scored and recorded manually. More than 60 thousand tracefiles and 6.9 million genotypes from the present report are publicly available via the internet (<ext-link ext-link-type="uri" ns0:href="http://cgemm.louisville.edu/USDA/index.html">http://cgemm.louisville.edu/USDA/index.html</ext-link>).</p><p>For mRNA transcript analysis, ovine blood (3 mL) was collected (Tempus Blood RNA tubes, Life Technologies Corporation, Carlsbad, CA, USA) and stored at &#8722;20&#176;C prior to RNA extraction. Whole blood RNA was purified by centrifugation and filtration according to the manufacture's protocol (Tempus Spin RNA isolation kits, Life Technologies Corporation). RNA quantity and quality were determined spectophotometrically (ND-1000, NanoDrop Technologies, Inc., Wilmington, DE, USA; and Agilent 2100 Bioanalyzer (Agilent Technologies, Inc., Santa Clara, CA, USA). The complete <italic>TMEM154</italic> mRNA coding region was amplified by PCR from cDNA (SuperScript III One-Step RT-PCR System, Platinum <italic>Taq</italic> High Fidelity, Invitrogen Corporation, Carlsbad, CA, USA). The 25 &#181;L reactions contained 1&#215; of the manufacturer's reagent cocktail, 0.2 &#181;M each of the sense and antisense primers (<xref ref-type="supplementary-material" rid="pgen.1002467.s004">Table S4</xref>), 0.5 &#181;L SuperScript III RT/Platinum <italic>Taq</italic> High Fidelity Enzyme Mix, and 30&#8211;50 ng of total RNA. Reaction conditions were the following: 1 cycle of cDNA synthesis at 55 &#176;C for 30 minutes followed by pre-denaturation at 94 &#176;C for 2 minutes; 40 cycles of PCR amplification at 94 &#176;C for 15 seconds, 58 &#176;C for 30 seconds, 68 &#176;C for 1 minute; and 1 cycle of final extension at 68 &#176;C for 5 minutes. As a control for DNA contamination and any putative <italic>TMEM154</italic> pseudogenes, duplicate sample reactions to those described above were subjected to PCR without preceding cDNA synthesis. Successful amplification of 1,012 bp fragments was monitored by gel electrophoresis. Amplicons were not observed in RT-PCR reactions lacking cDNA synthesis. Following an Exonuclease I digestion <xref ref-type="bibr" rid="pgen.1002467-Smith1">[54]</xref>, <italic>TMEM154</italic> RT-PCR amplicons were sequenced with dye-terminator chemistry and separated by capillary electrophoresis (ABI 3730, PE Applied Biosystems, Foster City, CA, USA). The oligonucleotide primers for PCR and sequencing are listed in <xref ref-type="supplementary-material" rid="pgen.1002467.s004">Table S4</xref>. Sequences were analyzed for polymorphisms and scored manually with Phred and Phrap <xref ref-type="bibr" rid="pgen.1002467-Ewing1">[55]</xref>, <xref ref-type="bibr" rid="pgen.1002467-Ewing2">[56]</xref>, Polyphred (version 6.10) <xref ref-type="bibr" rid="pgen.1002467-Nickerson1">[57]</xref> and Consed software <xref ref-type="bibr" rid="pgen.1002467-Gordon1">[53]</xref>.</p><p>An artiodactyl species panel of DNAs similar to that described previously <xref ref-type="bibr" rid="pgen.1002467-Laegreid1">[58]</xref> was sequenced to provide an estimate of the likely ancestral state of the polymorphic ovine <italic>TMEM154</italic> codons. This panel is composed primarily of species from the Pecoran clade, whose common ancestor dates to about 30 million years ago <xref ref-type="bibr" rid="pgen.1002467-Montgelard1">[59]</xref>. Oligonucleotide primers derived from ovine <italic>TMEM154</italic> genomic sequences were used in PCR assays to amplify exons 1 and 2 and PCR products for both exons were produced for the following species: Wyoming bighorn sheep (<italic>Ovis canadensis</italic>, n&#8202;=&#8202;7), American plains bison (<italic>Bison bison</italic>, n&#8202;=&#8202;7), Alaskan caribou (<italic>Rangifer tarandus</italic>, n&#8202;=&#8202;7) Wyoming elk (<italic>Cervus canadensis nelsoni</italic>, n&#8202;=&#8202;7), Texas exotic red deer (<italic>Cervus elaphus</italic>, n&#8202;=&#8202;2), Texas exotic fallow deer (<italic>Cervus dama</italic>, n&#8202;=&#8202;1), gaur (<italic>Bos gaurus</italic>, n&#8202;=&#8202;2), domestic goat (<italic>Capra hircus</italic>, n&#8202;=&#8202;4), Arkansas exotic water buffalo (<italic>Bubalus bubalis</italic>, n&#8202;=&#8202;l), Wyoming mule deer (<italic>Odocoileus hemionus</italic>, n&#8202;=&#8202;7), Wyoming white-tailed deer (<italic>Odocoileus virginianus</italic>, n&#8202;=&#8202;5), Wyoming mountain goat (<italic>Oreamnos americanus</italic> n&#8202;=&#8202;8), and Alaskan and Wyoming moose (<italic>Alces alces</italic>, n&#8202;=&#8202;8), for a total of 66 non-ovine artiodactyl individuals. To ensure that amplified DNA sequences were not derived from spurious ovine DNA, only those sequences with distinctive species-associated nucleotide differences were included in the analysis. Proteins encoded by Pecoran species were more than 95% identical to that encoded by ovine <italic>TMEM154</italic> haplotype 3.</p>