<p>HCT116, K562, A549 and HeLa cells were from ATCC. miR-34a<sup>+/+</sup> and miR-34a<sup>&#8722;/&#8722;</sup> MEFs were generated from E14.5 littermate embryos. A full description of the mice will be published elsewhere. MEFs were transformed by infecting the cells with retroviruses encoding H-RAS-V12 and E1A and by selection with puromycin (1 &#181;g/ml) and hygromycin (50 &#181;g/ml). The plasmids for expression of H-RAS-V12 (plasmid 9051) and E1A (plasmid 18748) were obtained from Addgene. The VSV-G pseudotyped viruses were produced in 293T cells using the standard protocol. MEFs, HCT116, A549 and HeLa cells were grown in DMEM with 10% fetal bovine serum and supplemented with penicillin, streptomycin, HEPES, L-glutamine and &#946;-mercaptoethanol, K562 cells were grown in RPMI containing 10% fetal bovine serum and the same supplements.</p><p>For most experiments, 2&#215;10<sup>6</sup> HCT116 or K562 cells were transfected with 200 pmol hsa-miR-34a or cel-miR-67 miRNA mimics (Dharmacon), using Amaxa nucleofection according to the manufacturer's protocol. Biotin was attached to the 3&#8242;-end of the active strand. HeLa and A549 cells were transfected with Lipofectamine 2000 and miRNA mimics at a final concentration of 50 nM (Invitrogen). To study the association of Bi-miRNAs with HA-Ago1 or HA-Ago2, pIRESNeo (Clontech) or pIRESNeo-HA-Ago1 or pIRESNeo-HA-Ago2 (Addgene) plasmids were co-transfected in six-well plates (2 &#181;g/well, 1&#215;10<sup>6</sup> cells/well) with 200 pmol Bi-miR-34a or Bi-cel-miR-67 using Amaxa as per the manufacturer's instructions.</p><p>Total RNA was isolated using Trizol reagent (Invitrogen), treated with DNase I (Ambion) and reverse transcribed using random hexamers and superscript III reverse transcriptase (Invitrogen). qRT-PCR was performed in triplicate samples using SYBR Green FastMix (Quanta) on a BioRad CFX96. mRNA levels were normalized to housekeeping genes <italic>GAPDH</italic>, <italic>UBC</italic> or <italic>SDHA</italic>. miRNA was quantified in triplicate using the TaqMan MicroRNA Assay (Applied Biosystems) as per the manufacturer's instructions and normalized to U6. Primer sequences are listed in <xref ref-type="supplementary-material" rid="pgen.1002363.s008">Table S3</xref>.</p><p>Whole cell lysates from transfected K562 or HCT116 cells were prepared using RIPA buffer. Proteins were analyzed by SDS-PAGE, transferred to nitrocellulose membranes and probed with the following antibodies: AXL [4566], ARAF [4432], MEK1 [9124], CDK4 [2906], MCM2 [3619], PKMYT1 [4282], PLK1 [4513], SMAD4 [9515], FOXP1 [2005], RBBP4 [4633], AKT [9272], pAKT ser-473 [4051], ERK [4370], pERK [9107] from Cell Signaling; MET [sc-161], MCM5 [sc-165995], E2F1 [sc-251], E2F3 [sc-879], CHEK1 [sc-8408] from Santa Cruz; ACSM3 [SAB1400253], MAD2L2 [SAB1400387], AGBL5 [AV53752], CCNG2 [AV03032], PSMD5 [WH0005711M1] from Sigma; MCM4 [06-1296] from Millipore; and PI3KR [610045], BD Biosciences. Western Blots were quantified by densitometry.</p><p>HCT116 or K562 cells (1&#215;10<sup>6</sup>) were transfected in triplicate with Bi-miR-34a or Bi-cel-miR-67 (Dharmacon) as described above and then cultured in six-well plates. Twenty-four hours later, the cells from 3 wells were pelleted at 500&#215;g. After washing twice with PBS, cell pellets were resuspended in 0.7 ml lysis buffer (20 mM Tris (pH 7.5), 100 mM KCl, 5 mM MgCl<sub>2</sub>, 0.3% NP-40, 50 U of RNase OUT (Invitrogen), complete mini-protease inhibitor cocktail (Roche Applied Science)), and incubated on ice for 5 min. The cytoplasmic lysate was isolated by centrifugation at 10,000&#215;g for 10 min. Streptavidin-coated magnetic beads (Invitrogen) were blocked for 2 hr at 4&#176;C in lysis buffer containing 1 mg/ml yeast tRNA and 1 mg/ml BSA (Ambion) and washed twice with 1 ml lysis buffer. Cytoplasmic lysate was added to the beads and incubated for 4 h at 4&#176;C before the beads were washed five times with 1 ml lysis buffer. RNA bound to the beads (pull-down RNA) or from 10% of the extract (input RNA), was isolated using Trizol LS reagent (Invitrogen). The level of mRNA in the Bi-miR-34a or Bi-cel-miR-67 control pull-down was quantified by qRT-PCR or mRNA microarray. For qRT-PCR, mRNA levels were normalized to a housekeeping gene (<italic>GAPDH</italic>, <italic>SDHA</italic> or <italic>UBC</italic>). The enrichment ratio of the control-normalized pull-down RNA to the control-normalized input levels was then calculated.</p><p>Total RNA (independently in two experiments) was amplified, labeled and hybridized to Affymetrix U133 plus 2.0 mRNA microarrays. The quality of the RNA was assessed before performing the microarray and the quality of the microarray data was assessed using affyPLM and Affy software. The replicate data sets for the 4 sets of samples (pull-down and input for miR-34a and cel-miR-67) were compared using an unsupervised hierarchical clustering algorithm, which verified the similarity of the duplicates. The microarray data were normalized using RMA <xref ref-type="bibr" rid="pgen.1002363-Chang1">[7]</xref> to reduce interarray variation. The enrichment ratio {Bi-miR-34a PD/Bi-cel-miR-67 PD}/{Bi-miR-34a input/Bi-cel-miR-67 input} was calculated for each probe. For genes represented by multiple probes, the mean ratio for all the probes was calculated. Genes for which none of the probe hybridization signals exceeded the background were considered not expressed and were disregarded in the analysis. For informatic analysis of the PD data, genes whose enrichment ratio were &#8805;1 SD above background based on a log-normal distribution were considered &#8220;hits&#8221;.</p><p>HCT116 or K562 cells were transfected in independent duplicate experiments as above with unbiotinylated miR-34a or cel-miR-67 (Dharmacon) and total RNA was harvested 24 hr later and analyzed as above by gene expression microarrays. After normalization, fold changes for each probe were calculated as the ratio of input RNA from miR-34a-transfected cells to the ratio of input RNA from cel-miR-67-transfected cells. Genes were considered down-regulated if the ratio decreased by at least 20%, which corresponded to &#8764;1 SD. To test the expression levels of putative target sets, each gene list was plotted in a cumulative distribution function (CDF) plot, and the Kolmogorov-Smirnov [KS] test was used for statistical comparisons between gene sets.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To determine whether a gene was also a predicted target of miR-34a, the presence of miR-34a binding sites was analyzed using TargetScan 4.2 (<ext-link ext-link-type="uri" ns0:href="http://www.targetscan.org/">http://www.targetscan.org/</ext-link>) <xref ref-type="bibr" rid="pgen.1002363-Lewis1">[39]</xref>, <xref ref-type="bibr" rid="pgen.1002363-Lewis2">[54]</xref>, <xref ref-type="bibr" rid="pgen.1002363-Friedman1">[55]</xref> or PITA (<ext-link ext-link-type="uri" ns0:href="http://132.77.150.113/pubs/mir07/mir07_prediction.html">http://132.77.150.113/pubs/mir07/mir07_prediction.html</ext-link>) <xref ref-type="bibr" rid="pgen.1002363-Kertesz1">[50]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The mature hsa-miR-34a sequence was obtained from miRBase (<ext-link ext-link-type="uri" ns0:href="http://mirbase.org/">http://mirbase.org/</ext-link>). All RefSeq human mRNA sequences were downloaded from NCBI in July 2009 (<ext-link ext-link-type="uri" ns0:href="http://ftp.ncbi.nih.gov/">http://ftp.ncbi.nih.gov/</ext-link>). mRNAs were indexed by Entrez Gene ID; in cases where multiple sequences matched a gene ID, the sequence with the longest 3&#8242;UTR was selected. For each test gene list and miR-34a hexamer, the miR-34a hexamer frequency (hexamer matches per kb of sequence) was calculated. The frequency of hexamer matches for all genes on the microarray (the background set) was also determined. Gene IDs with no corresponding sequence in the database were excluded from analysis. Monte Carlo simulations of equally sized random gene sets (without replacement) were used to generate an empirical 2-tailed p-value for each gene set/hexamer combination. When p&lt;1E-4, the p-value was calculated from curve fitting relative to the random background distribution.</p><p>For each of the lists of down-regulated and pull-down-enriched genes, the p-value of over-representation in a suite of canonical pathways (KEGG <xref ref-type="bibr" rid="pgen.1002363-Kanehisa1">[56]</xref> and Wikipathways <xref ref-type="bibr" rid="pgen.1002363-Pico1">[57]</xref>) was determined using the hypergeometric distribution. A visualization of the relationship between the enriched pathways (p&lt;0.001) based on the number of overlapping genes was rendered using Cytoscape <xref ref-type="bibr" rid="pgen.1002363-Shannon1">[58]</xref>. The network of gene-gene interactions underlying these relationships was constructed based on interactions supplied by MetaCore (GeneGo Inc). Physical, predicted and genetic interactions were used to connect the down-regulated and pull-down enriched genes within the significant signaling, cell cycle or DNA repair pathways. Signaling pathway genes with no connection to any other node were removed and the network was arranged according to predicted sub-cellular localization.</p><p>HeLa cells were cotransfected in 24 well plates using Lipofectamine 2000 (Invitrogen) with 50 nM miR-34a mimic or control miRNA mimic and 50 ng of psiCHECK2 (Promega) vector containing the MRE or 3&#8242;UTR of indicated genes cloned into the multiple cloning site of <italic>Renilla</italic> luciferase. After 48 hr of transfection (unless otherwise indicated) luciferase activities were measured using the Dual Luciferase Assay System (Promega) and Top count NXT microplate reader (Perkin Elmer) per manufacturer's instructions. All experiments were performed at least in triplicate. Results were normalized to those obtained in cells transfected with an empty vector. For some experiments, a perfectly complementary antisense sequence to the active strand of miR-34a was inserted into the multiple cloning site for use as a positive control. Data were normalized to <italic>Firefly</italic> luciferase and results from 3 independent experiments were compared. Sequence of primers used for cloning 3&#8242;UTRs for miR-34a target genes are listed in <xref ref-type="supplementary-material" rid="pgen.1002363.s009">Table S4</xref>. MREs sequences were cloned into psiCHECK-2 by annealing complementary oligomers matching each MRE sequence (<xref ref-type="supplementary-material" rid="pgen.1002363.s004">Figures S4</xref>, <xref ref-type="supplementary-material" rid="pgen.1002363.s005">S5</xref>) with overhanging ends complementary to the XhoI and NotI sites of psiCHECK-2.</p><p>HCT116, HeLa and A549 cells were transfected as described above. One day after transfection, cells were placed in serum-free medium or medium containing 10% fetal calf serum. 48 hours after the medium was changed, total cell numbers were counted. MEFs were plated at a density of 2.5&#215;10<sup>5</sup> or 5&#215;10<sup>5</sup> cells per well of a 6-well plate. The medium was changed to vary serum concentration 24 hr after plating. The MEFs were harvested 24 hr later and counted using Trypan blue staining or stained in PBS+0.4% BSA with annexinV-APC (Invitrogen) at a 1&#8758;30 dilution, then washed once and stained with propidium iodide (4 &#181;g/ml) (Sigma-Aldrich).</p>