<p>Included in this consortium project were 19,091 cases and 20,606 controls from 14 studies (<xref ref-type="table" rid="pgen-1002532-t001">Table 1</xref>). Detailed descriptions of these participating studies and demographic characteristics of study participants are provided in <xref ref-type="supplementary-material" rid="pgen.1002532.s012">Text S1</xref>. Briefly, the consortium included 23,981 Chinese women, 11,907 Korean women, 3,809 Japanese women. The Chinese women were from 8 studies: Shanghai [n&#8202;=&#8202;13,642, Shanghai Breast Cancer Study, Shanghai Breast Cancer Survival Study (SBCSS), Shanghai Endometrial Cancer Study (SECS), Shanghai Women Health Study (SWHS)] <xref ref-type="bibr" rid="pgen.1002532-Zheng1">[8]</xref>, , Nanjing (n&#8202;=&#8202;3,623) <xref ref-type="bibr" rid="pgen.1002532-Liang1">[27]</xref>, Tianjin (n&#8202;=&#8202;2,882) <xref ref-type="bibr" rid="pgen.1002532-Zhang2">[28]</xref>, Taiwan (n&#8202;=&#8202;2,131) <xref ref-type="bibr" rid="pgen.1002532-Ding1">[29]</xref>, and Guangzhou (n&#8202;=&#8202;1,703). The Korean women were from four studies [Seoul Breast Cancer Study (SeBCS) (n&#8202;=&#8202;6,292) <xref ref-type="bibr" rid="pgen.1002532-Choi1">[30]</xref>, Korea NCC (n&#8202;=&#8202;1,009), KoGES (n&#8202;=&#8202;3,209) <xref ref-type="bibr" rid="pgen.1002532-Cho1">[31]</xref>, and KOHBRA (n&#8202;=&#8202;1,397) <xref ref-type="bibr" rid="pgen.1002532-Han1">[32]</xref>]. The Japanese women were from three studies conducted in Hawaii and Los Angeles [n&#8202;=&#8202;1,719; Multiethnic Cohort Study (MEC) <xref ref-type="bibr" rid="pgen.1002532-Kolonel1">[33]</xref>], Nagoya (n&#8202;=&#8202;1,288) <xref ref-type="bibr" rid="pgen.1002532-Hamajima1">[34]</xref>, and Nagano (n&#8202;=&#8202;802) <xref ref-type="bibr" rid="pgen.1002532-Itoh1">[35]</xref> (<xref ref-type="table" rid="pgen-1002532-t001">Table 1</xref>). Approval was granted from relevant institutional review boards in all study sites; all included subjects gave informed consent.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The Genotyping protocol for Stage I has been described previously <xref ref-type="bibr" rid="pgen.1002532-Zheng1">[8]</xref>. Briefly, the initial 300 subjects were genotyped using the Affymetrix GeneChip Mapping 500K Array Set. The remaining 4,985 subjects were genotyped using the Affymetrix Genome-Wide Human SNP Array 6.0. We included one negative control and at least three positive quality control (QC) samples from the Coriell Cell Repositories (<ext-link ext-link-type="uri" ns0:href="http://ccr.coriell.org/">http://ccr.coriell.org/</ext-link>) in each of the 96-well plates for Affymetrix SNP Array 6.0 genotyping. A total of 273 positive QC samples were successfully genotyped, and the average concordance rate was 99.9% with a median value of 100%. The sex of all study samples was confirmed to be female. Genetically identical, unexpected duplicated samples were excluded, as were close relatives with a pair-wise proportion of identify-by-descent (IBD) estimate greater than 0.25. All samples with a call rate&lt;95% were excluded. The SNPs were excluded if: (i) MAF&lt;1%, (ii) call rate&lt;95%, or (iii) genotyping concordance rate&lt;95% in quality control samples. The final dataset included 2,918 cases and 2,324 controls for 690,947 markers. There are 21,223 SNPs that were on Affymetrix 500K Array Set but not on the Affymetrix SNP Array 6.0. These SNPs were excluded. SNPs on the Affymetrix 6.0 array but not on the Affymetrix 500k array were treated as missing data for those samples genotyped on using the Affymetrix 500k array. Similar results were obtained after excluding women genotyped by Affymetrix 500K Array Set from the analyses.</p><p>Genotyping for Stage IIa was completed using the Illumina iSelect platform. To compare the consistency between the Affymetrix and Illumina iSelect platforms, we also included 43 samples from Stage I that were genotyped by Affymetrix SNP 6.0. Similar to the QC procedures used in Stage I, the following criteria were used to exclude samples: (i) call rate&lt;95%; or (ii) unexpected duplicated samples based on IBD estimate. SNPs were excluded if: (i) call rate&lt;95%, or (ii) genotyping concordance rate&lt;95% in quality control samples when compared with Affymetrix 6.0 data. After QC, the mean concordance rate was 99.85% between Illumina iSelect and Affymetrix 6.0 genotyping.</p><p>Data for the SNPs analyzed in Stage IIb were extracted from the Korean GWAS genotyped using the Affymetrix Genome-Wide Human SNP Array 6.0 chip. A total of 30 QC samples were successfully genotyped, and the concordance rate was 99.83%. The sex of all samples was confirmed to be female. The SNPs were excluded if: (1) genotype call rate&lt;95%, (2) MAF&lt;1% in either the cases or controls, (3) deviation from HWE at <italic>P</italic>-value&lt;10<sup>&#8722;6</sup>, and (4) poor cluster plot in either the cases or controls.</p><p>Genotyping for Stage III and all samples from Koreans in Stage IV was completed using the iPLEX Sequenom MassArray platform in the Vanderbilt Molecular Epidemiology Laboratory. Included in each 96-well plate as QC samples were one negative control (water), two blinded duplicates, and two samples from the HapMap project. To compare the consistency between the Affymetrix and Sequenom platforms, we also genotyped 45 samples included in Stage I. The mean concordance rate was 99.67% for the blind duplicates, 98.88% for HapMap samples, and 99.52% between Sequenom and Affymetrix 6.0 genotyping. Data quality from the Hong Kong study was low and thus data from the study were excluded for the current analysis. Genotyping for two Chinese studies (Nanjing and Guangzhou) in Stage IV was completed using the iPLEX Sequenom MassArray platform at the Fudan University, Shanghai, China. Blind duplicate QC samples were included and the mean concordance rate was 98.70%. Genotyping for the Tianjin study in Stage IV was performed using TaqMan assays. Genotyping assay protocols were developed and validated at the Vanderbilt Molecular Epidemiology Laboratory, and TaqMan genotyping assay reagents were provided to investigators of the Tianjin study (Tianjin Cancer Institute and Hospital). For the MEC study, data for the three SNPs presented in this study were extracted from the GWA scan data generated using Illumina 660W. For SNPs not included on the chip, imputed data using HapMap as reference were extracted. Genotype frequencies for SNP rs9485372 deviated from HWE in controls (P&#8202;=&#8202;0.004), therefore, this SNP was excluded in data analyses. Not all SNPs for Stage IV were genotyped in all studies included in Stage IV due to genotyping failure or the use of different genotyping platforms (<xref ref-type="supplementary-material" rid="pgen.1002532.s011">Table S8</xref>).</p><p>SNP selection for Stage II replication: Promising SNPs were selected for replication in Stage II based on the following criteria: 1) minor allele frequency (MAF)&#8805;5%; 2) <italic>P</italic>&lt;0.02 in Stage I; 3) Hardy-Weinberg equilibrium (HWE) test <italic>P</italic>&gt;1.0&#215;10<sup>&#8722;6</sup> in controls; 4) not in strong linkage disequilibrium (LD) (r<sup>2</sup>&lt;0.5) with any of the previously confirmed breast cancer genetic risk variants or SNPs evaluated in our previous studies <xref ref-type="bibr" rid="pgen.1002532-Zheng1">[8]</xref>, <xref ref-type="bibr" rid="pgen.1002532-Long1">[12]</xref>; and 5) high genotyping quality as indicated by very clear genotyping clusters checked manually. When multiple SNPs are in LD with r<sup>2</sup>&#8805;0.5, one SNP with the lowest <italic>P</italic>-value was selected. In total, 6,303 SNPs were selected for replication. A total of 5,906 SNPs (93.7%) were successfully designed by Illumina and included in the iSelect array. After stringent QC procedures, data from 5,365 SNPs were considered high quality for association analyses in Stage IIa, which include 1,613 breast cancer patients and 1,800 controls recruited from Shanghai studies.</p><p>SNP selection for Stage III replication: Among the 5,365 SNPs successfully genotyped in Stage IIa, 68 SNPs were selected for Stage III replication in an independent set of 5,203 cases and 5,138 controls recruited from Shanghai and several other East Asian populations (<xref ref-type="table" rid="pgen-1002532-t001">Table 1</xref> and <xref ref-type="supplementary-material" rid="pgen.1002532.s012">Text S1</xref>). The selection criteria are: 1) an association with breast cancer risk in Stage IIa with P&#8804;0.05; 2) the direction of the association consistent in both stages; and 3) P&#8804;0.001 in the merged data of Stage I and IIa. During the course of Stage III genotyping, genome-wide association scan data from 2,359 cases and 2,052 controls were obtained from the Seoul Breast Cancer GWAS (Stage IIb). Therefore, we performed a meta-analysis of Stage IIa and IIb data. Of the 5,297 SNPs which were not selected initially for Stage III replication based on Stage IIa data alone, data were available for 4,913 SNPs in Stage IIb. Meta-analyses of these 4,913 SNPs from Stage IIa and IIb yielded 26 additional SNPs that showed an association at <italic>P</italic>&#8804;0.05 and in the same direction among stages I, IIa, and IIb. These 26 SNPs were then added to the list of SNPs to be genotyped in Stage III.</p><p>SNP selection for Stage IV replication: Based on the results of the first three stages, 22 top SNPs were selected for Stage IV evaluation and genotyped in up to 17,423 additional subjects (7,489 cases and 9,934 controls) (<xref ref-type="table" rid="pgen-1002532-t001">Table 1</xref> and <xref ref-type="supplementary-material" rid="pgen.1002532.s012">Text S1</xref>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Case-control differences in selected demographic characteristics and major risk factors were evaluated using t-tests (for continuous variables) and Chi-square tests (for categorical variables). Associations between SNPs and breast cancer risk were assessed using odds ratios (ORs) and 95% confidence intervals (CIs) derived from logistic regression models. ORs were estimated for heterozygote and homozygote for the variant allele compared with homozygotes for the common allele. ORs were also estimated for the variant allele based on a log-additive model and adjusted for age, and study site, when appropriate. Stratified analyses by ethnicity, menopausal status, and estrogen receptor (ER) status were carried out. PLINK version 1.06 was used to analyze genome-wide data obtained in Stage I and the replication data in Stage IIa. Results from Stage IIb were also obtained from PLINK version 1.06. Meta-analyses of Stage IIa and Stage IIb were performed using a weighted z-statistics method, where weights were proportional to the square root of the number of individuals in each sample and standardized such that the weights added up to one. The z-statistic summarizes the magnitude and direction of the effect relative to the reference allele. An overall z-statistic and p value were then calculated from the weighted average of the individual statistics. Calculations were implemented in the METAL package (<ext-link ext-link-type="uri" ns0:href="http://www.sph.umich.edu/csg/abecasis/Metal">http://www.sph.umich.edu/csg/abecasis/Metal</ext-link>). Individual data were obtained from each study for Stage IV SNPs for a pooled analysis, which were conducted using SAS, version 9.2, with the use of two-tailed tests.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We first investigated the population structure by estimating inflation factor &#955; using all 690,947 SNPs SNPs that passed the QC. The inflation factor &#955; was estimated to be 1.042, suggesting that any population substructure, if present, should not have any appreciable effect on the results. Among the final 690,947 SNPs obtained in Stage I after QC, we generated a list of 196,471 SNPs with pairwise LD&lt;0.2 by using plink (<ext-link ext-link-type="uri" ns0:href="http://pngu.mgh.harvard.edu/~purcell/plink/">http://pngu.mgh.harvard.edu/~purcell/plink/</ext-link>). Then, principal components were estimated based on these 196,471 SNPs using EIGENSTRAT <xref ref-type="bibr" rid="pgen.1002532-Price1">[36]</xref>. We then drew a plot for all Stage I and HapMap II subjects based on the first two principal components (<xref ref-type="fig" rid="pgen-1002532-g004">Figure 4</xref>). All study participants in Stage I were clustered very closely with HapMap Asians. The first 5 or 10 principal components were adjusted in the logistic regression analyses for evaluating associations of SNPs and breast cancer risk.</p><p>A: all individuals from Stage I and HapMap; B: breast cancer cases and controls from Stage I.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To evaluate the combined effect of SNPs located in chromosome 6q25.1 on breast cancer risk, we created a genetic risk score (GRS) by summing the number (0&#8211;2) of risk alleles that each woman carried for each of the three SNPs, including rs9383951, rs9485372, rs2046210. The GRS was constructed among those who had complete data for all three SNPs. We also did imputation using MACH (<ext-link ext-link-type="uri" ns0:href="http://www.sph.umich.edu/csg/abecasis/MACH/index.html">http://www.sph.umich.edu/csg/abecasis/MACH/index.html</ext-link>) with HapMap II Asian data as reference. LD structure was estimated from the flanking 100 kb of these three SNPs and the <italic>ESR1</italic> gene using data from HapMap II Asians (<xref ref-type="supplementary-material" rid="pgen.1002532.s001">Figure S1</xref>). All SNPs in the LD blocks including rs9485372, rs2046210 and rs9383951 and SNPs inside the <italic>ESR1</italic> gene were analyzed in relation to breast cancer risk with age, rs9485372, rs9383951 and rs2046210 adjusted.</p>