<p>
<italic>Drosophila</italic> stocks containing the following transgenes and mutant alleles were used: UAS-<italic>org-1</italic> (gift of G. Pflugfelder, Univ. Wurzburg, Germany), attP40 and <italic>nos</italic>-phiC31intNLS <xref ref-type="bibr" rid="pgen.1002531-Bischof1">[115]</xref> (gift of N. Perrimon, Harvard University, USA), <italic>lbl</italic>-<italic>lacZ</italic> and <italic>Ndg</italic>-<italic>lacZ</italic>
<xref ref-type="bibr" rid="pgen.1002531-Philippakis1">[5]</xref>, <italic>ap-lacZ</italic> (gift of J. Botas, Baylor College, USA) <xref ref-type="bibr" rid="pgen.1002531-Capovilla1">[35]</xref>, <italic>acj6<sup>6</sup></italic> (gift of J. Carlson, Yale University, USA) <xref ref-type="bibr" rid="pgen.1002531-Clyne1">[116]</xref>, and <italic>twi</italic>-<italic>Gal4</italic>
<xref ref-type="bibr" rid="pgen.1002531-Carmena1">[49]</xref>.</p><p>Enhancer regions were either synthesized <italic>in vitro</italic> (Integrated DNA Technologies, Coralville, IA, USA) or PCR-amplified and sequence-verified and then subcloned into the reporter vector pWattB-GFP (B. W. Busser, L. Shokri, S. A. Jaeger, S. S. Gisselbrecht, A. Singhania, M. F. Berger, B. Zhou, M. L. Bulyk and A. M. Michelson, unpublished data) or pWattB-nlacZ. The pWattB-nlacZ vector was constructed by cloning the EcoRI-SpeI fragment from a version of pH-pelican <xref ref-type="bibr" rid="pgen.1002531-Barolo1">[117]</xref> in which nuclear lacZ replaced cytoplasmic lacZ into the EcoRI-SpeI sites in the pWattB-GFP vector. All constructs were targeted to attP40 <xref ref-type="bibr" rid="pgen.1002531-Markstein2">[47]</xref> with phiC31-mediated integration <xref ref-type="bibr" rid="pgen.1002531-Groth1">[46]</xref>, and homozygous viable insertion lines were obtained. Whole-embryo immunohistochemistry, <italic>in situ</italic> hybridization and fluorescent <italic>in situ</italic> hybridization with tyramide signal amplification (Invitrogen, Carlsbad, CA, USA) followed standard protocols <xref ref-type="bibr" rid="pgen.1002531-Halfon1">[28]</xref>. Embryo collections for <italic>twi</italic>-<italic>Gal4</italic> UAS-<italic>org-1</italic> were incubated at 25&#176;C. For fluorescent staining, the following antibodies were used: mouse anti-Ladybird early (Lbe) (1&#8758;2500, gift of K. Jagla; Lbe and Lbl are co-expressed in the same mesodermal cells), rabbit anti-Slou (1&#8758;200, gift of M. Baylies), chicken anti-GFP (1&#8758;2000, Abcam, Cambridge, MA), mouse anti-&#946;gal (1&#8758;500, Promega, Madison, WI), rabbit anti-Kirre (1&#8758;200, gift of K. Fischbach), rabbit anti-MHC (1&#8758;500, gift of D. Kiehart), and guinea pig anti-Eve (1&#8758;200, gift of D. Kosman).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Embryo RNAi was performed as previously described <xref ref-type="bibr" rid="pgen.1002531-Estrada1">[22]</xref>. Using SnapDragon (<ext-link ext-link-type="uri" ns0:href="http://www.flyrnai.org/cgi-bin/RNAi_find_primers.pl">http://www.flyrnai.org/cgi-bin/RNAi_find_primers.pl</ext-link>), two independent gene segments for synthesis of <italic>org-1</italic> double-stranded RNA (dsRNA) were selected with lengths of 570 and 473 bp and with less than 20 and 22 bp of identity to any other predicted gene, respectively. These segments of the <italic>org-1</italic> gene were PCR-amplified from primary embryonic cDNA using the primers, <named-content content-type="gene">CGTCCAAAAAGTTCAAGGGA</named-content> and <named-content content-type="gene">GCTCGTTCTCATCCAAGGAG</named-content> (570 bp) and <named-content content-type="gene">GCTCCAACAGAGCCAGAATC</named-content> and <named-content content-type="gene">CCGAACCGTAAAAACTTGGA</named-content> (473 bp), and transcribed <italic>in vitro</italic> using the MEGAscript RNAi kit (Ambion, USA). <italic>lbl-GFP</italic>, <italic>slou-GFP</italic> or <italic>MHC-tauGFP</italic> embryos were dechorionated and injected with negative control (<italic>lacZ</italic>) or <italic>org-1</italic> dsRNA at the syncytial blastoderm stage and allowed to develop to stage 14 or 16 before examination by fluorescence microscopy for assessment of reporter GFP expression. A similar protocol was used to assess the function of <italic>acj6</italic> in <italic>Ndg-GFP</italic> embryos.</p><p>The sixteen sequences in the training set of <italic>Drosophila melanogaster</italic> FC enhancers range in length from 311 to 2068 bp (average length 1232 bp), in GC-content from 39% to 49% (average GC-content 43%), and in repeat-content from 0% to 7% (average 1.5%). The twenty-four orthologs have similar characteristics, with an average length of 1311 bp, GC-content of 43% and repeat content of 5%. The control set comprised 1000 randomly selected <italic>D. melanogaster</italic> noncoding genomic sequences with length, GC- and repeat-content matching those of the enhancer set.</p><p>Putative TFBSs were identified by searching the sequences with MAST <xref ref-type="bibr" rid="pgen.1002531-Bailey1">[118]</xref> for motifs in TRANSFAC Release 2009.2 <xref ref-type="bibr" rid="pgen.1002531-Matys1">[119]</xref>, in addition to binding sequences for Tin, Twi and Pnt from the literature <xref ref-type="bibr" rid="pgen.1002531-Philippakis1">[5]</xref>. MAST was run independently on each individual sequence with default setup and parameters. In particular, for the final analysis of the TFBS composition of FC enhancers, we examined the sequences for the occurrence of 11 types of TFBSs: POUHD (V$OCT_01, V$POU1F1_Q6, V$OCT4_02), Tbx (V$TBX5_01, I$BYN_Q6), Myb (V$MYB_Q6), Fkh (V$FOXO3_01, V$FOXO1_Q5, V$FREAC2_01), HD (I$ABDA_Q6, V$CDX5_Q5, V$IFP_03, V$PAX4_02), and Mef2 (V$AMEF2_Q6, V$HMEF2_Q6) and binding sites for Tcf, Mad, Ets, Twi, Tin <xref ref-type="bibr" rid="pgen.1002531-Philippakis1">[5]</xref>. Since the Position Weight Matrices (PWMs) for Tcf, Mad, Ets, Twi, and Tin were constructed from only a small number of sequences and we obtained few significant matches using MAST, we re-inspected the sequences manually, searching for the known binding sites of these TFs.</p><p>Over-represented TFBSs were determined by comparing the occurrence of the motifs among query sequences and background genomic sequence, and applying Fisher's exact test. We used a P-value threshold of 0.05. When indicated, we adjusted the P-values for multiple testing using the procedure suggested by Benjamini and Hochberg <xref ref-type="bibr" rid="pgen.1002531-Benjamini1">[120]</xref>.</p><p>Each nucleotide sequence in the FC enhancer and control sets was represented by the number of putative TFBSs per base pair. Putative TFBSs were identified by searching the sequences for motifs derived from three different sources: (1) 892 TF binding specificities characterized in TRANSFAC, (2) 3 binding sequences for Tin, Twi, and Pnt from the literature <xref ref-type="bibr" rid="pgen.1002531-Philippakis1">[5]</xref>, and (3) at most 50 motifs that PRIORITY <xref ref-type="bibr" rid="pgen.1002531-Narlikar2">[51]</xref> found to be over-represented in the training set of FC enhancers. In order to prevent the over-representation of motifs in <italic>D. melanogaster</italic> enhancer sequences with a large number of orthologs, <italic>de novo</italic> motifs were identified on a restricted set of 34 sequences, including at most two randomly selected orthologs for each <italic>D. melanogaster</italic> FC enhancer. Features relevant for distinguishing between enhancer sequences and controls were identified using linear support vector machine (SVM). We used a standard ten-fold cross-validation procedure to assess the accuracy of the classifier. In each fold of the cross-validation procedure, the <italic>de novo</italic> motifs were extracted using the training data, only, thereby ensuring that the test data were completely unseen before the predictions were made. The cross-validation procedure should help to prevent overfitting of the classifier.</p><p>We used Support Vector Machines <xref ref-type="bibr" rid="pgen.1002531-Chang1">[121]</xref> with a linear kernel, which only requires setting the penalization coefficient C. The performance of the SVM was evaluated using the area under the receiver operating characteristic (ROC) curve, which yields values between 0.5 (for a completely random guess) and 1.0 (for a perfect classifier). In order to compensate for the data unbalance, FC enhancer sequences and controls were assigned different misclassification costs (SVM soft-margin constants), giving equal overall weight to each class.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Given a training set of instances <italic>x<sub>1</sub>, &#8230;, x<sub>l</sub></italic>
<inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002531.e001.jpg" /></inline-formula> with associated labels <italic>y<sub>1</sub>, &#8230;, y<sub>l</sub></italic>
<inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002531.e002.jpg" /></inline-formula>, linear SVM solves the optimization problem <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002531.e003.jpg" /></inline-formula> subject to <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002531.e004.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002531.e005.jpg" /></inline-formula>
<xref ref-type="bibr" rid="pgen.1002531-ShaweTaylor1">[122]</xref>. Thus, after obtaining a linear SVM model, the weight vector <italic>w</italic> can be used to decide the relevance of each feature <xref ref-type="bibr" rid="pgen.1002531-Guyon1">[123]</xref>. The larger <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002531.e006.jpg" /></inline-formula>, the more important role of feature j in the decision function. We rank features&#8212;in our case, motifs&#8212;according to <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002531.e007.jpg" /></inline-formula>. For this purpose, we trained a classifier for 100 random partitions of the training data (containing two thirds of the total training data), computed the ranking for each feature, and finally ranked the features according to their median ranking.</p><p>We scanned the whole-genome of <italic>D. melanogaster</italic> (BDGP Release 5 assembly) with a sliding window of length 1000 base pairs and overlaps of 500 base pairs. The length of the window corresponds approximately to the average length of the sequences in the training data set (1280 base pairs). We scored 137,364 sequences after excluding sequences which overlap annotated coding regions by at least 50%. The cut-off for the genome scan was defined to obtain a false positive rate (FPR) of at most 5% by training and testing 100 classifiers on random partitions of the training data (containing two thirds of the total training data).</p><p>TF annotation for PWMs was obtained from TRANSFAC and the Broad Institute MsigDB database <xref ref-type="bibr" rid="pgen.1002531-Subramanian1">[124]</xref>.</p><p>To understand the effects of a sample size on the classifier performances, we first extracted a dataset of <italic>Drosophila melanogaster</italic> 62 enhancers active in mesoderm from the REDfly database <xref ref-type="bibr" rid="pgen.1002531-Gallo1">[32]</xref>. REDfly contains 176 partially overlapping enhancers active in mesoderm. To eliminate this redundancy, we clustered together overlapping sequences, and subsequently selected the shortest enhancer sequence from each cluster; sequences longer than 2 kb as well as sequences overlapping our dataset of enhancers active in muscle founder were excluded from the final dataset. Orthologs were selected randomly among sequences from 15 insect species <xref ref-type="bibr" rid="pgen.1002531-Siepel1">[125]</xref> with nucleotide identity ranging from 50 to 80%, so that at most two orthologs were selected for each <italic>Drosophila melanogaster</italic> enhancer in REDfly. As controls, we randomly sampled for each enhancer (and orthologs) 10 non-coding sequences from the <italic>Drosophila melanogaster</italic> genome with similar length, GC- and repeat-content.</p><p>Each nucleotide sequence in the enhancer and control sets was represented by the number of putative TFBSs per base pair. Putative TFBSs were identified by searching the sequences with MAST <xref ref-type="bibr" rid="pgen.1002531-Bailey1">[118]</xref> for motifs derived from two sources: (1) 892 TF binding specificities characterized in TRANSFAC Release 2009.2 <xref ref-type="bibr" rid="pgen.1002531-Matys1">[119]</xref>, and (2) 3 binding sequences for Tin, Twi, and Pnt from the literature <xref ref-type="bibr" rid="pgen.1002531-Philippakis1">[5]</xref>. Features relevant for distinguishing between enhancer sequences and controls were identified using linear SVM.</p>