<p>The Ins-CMV-C-B-A vector <xref ref-type="bibr" rid="pgen.1002399-Devlin1">[38]</xref> was modified to include a Gateway conversion cassette (Invitrogen) in the multiple cloning site, which includes a ccdB cassette flanked by attR1 and attR2 recombination sequences. A Gateway LR clonase reaction was used to insert a PCR amplicon of the GARS cDNA (human) into the pINS2-Gateway vector. The cDNA amplicon that was inserted into pINS2 included 47 basepairs of the endogenous GARS 5&#8242; UTR upstream of the start site of the mitochondrial isoform, and 19 basepairs of endogenous 3&#8242;UTR flanked by 5&#8242; and 3&#8242; untranslated regions (UTR) from rabbit &#946; globin also containing a polyadenylation signal at the 3&#8242; end. The cDNA and control elements are flanked by two chicken &#946;-globin 5&#8242;HS4 insulators, which have been shown to reduce variability in transgene expression caused by position effects of insertion site <xref ref-type="bibr" rid="pgen.1002399-Potts1">[39]</xref>. Pronuclear injection of this construct into single cell FVB/N embryos was performed in the NHGRI Transgenic Mouse Core according to existing protocols <xref ref-type="bibr" rid="pgen.1002399-Nagy1">[40]</xref>.</p><p>All mouse husbandry and procedures were conducted according to the NIH Guide for Care and Use of Laboratory Animals and were approved by NINDS or The Jackson Laboratory Animal Care and Use Committee. Tail, toe or ear tissue was lysed with proteinase K incubation at 55&#176;C and boiled for 10 min to inactivate the proteinase K. DNA derived from the mouse tissue was then used in PCR to determine genotype.</p><p>Primers GARS WT Tg F (<named-content content-type="gene">5&#8242;-CCCATTACTGGAAATGATCTA-3&#8242;</named-content>) and GARS WT Tg R (<named-content content-type="gene">5&#8242;-TTTCCGAGCGGACTGTCCGC-3&#8242;</named-content>), which anneal to exons 7 and 9, respectively, were used to determine if the wild-type transgene was present. Primers targeting <italic>Gars</italic> intron 6F (<named-content content-type="gene">5&#8242;-GCCTTGTTCTGTAACGTTTGCAC-3&#8242;</named-content>) and a primer specific to the <italic>Nmf249</italic> allele (<named-content content-type="gene">5&#8242;-CCAGGCATATTTCCTCCATATTT-3&#8242;</named-content>) were used to identify mice with the <italic>Nmf249</italic> mutation <xref ref-type="bibr" rid="pgen.1002399-Seburn1">[25]</xref>. Primers Gars C201R F (<named-content content-type="gene">5&#8242;-CACGTGCTTGCTCTAGCAAGA-3&#8242;</named-content>) and Gars C201R R (<named-content content-type="gene">5&#8242;-GTCTACCACTGAACACAGTCC-3&#8242;</named-content>) were used in PCR reaction, and the product was digested with <italic>HhaI</italic> restriction enzyme. PCR products containing the pathogenic mutation are digested into two smaller bands <xref ref-type="bibr" rid="pgen.1002399-Achilli1">[26]</xref>. Primers BgeoR (<named-content content-type="gene">5&#8242;-CGCCAGGGTTTTCCCAGT-3&#8242;</named-content>) and En2-i1F (<named-content content-type="gene">5&#8242;-AATGCCCAACACTTGTATGG-3&#8242;</named-content>) were used to determine if the <italic>XM256</italic> gene trap allele was present. To screen for <italic>XM256/XM256</italic> mice, primers flanking the gene trap insertion 519 bp into intron 2 of <italic>Gars</italic>, GarsXMvsWT2F (<named-content content-type="gene">5&#8242;-GCTTCCGCACTACCTGAACCCAAACT-3&#8242;</named-content>) and GarsXMvsWT2R (<named-content content-type="gene">5&#8242;-TGAATTCAGCAGCCCCCTCTGTACCC-3&#8242;</named-content>), were used. No <italic>XM256/XM256</italic> mice were identified, with or without the transgene. PCR amplification indicated the wild-type allele was present. All genotyping products were resolved on 2% agarose gels with ethidium bromide (Sigma, St. Louis, MO). <italic>Nmf249</italic> mice are maintained on a C57BL/6 background and are occasionally outcrossed to CAST/Ei to improve their ability to breed <xref ref-type="bibr" rid="pgen.1002399-Stum1">[21]</xref>. <italic>C201R</italic> mice were originally on a mixed C57BL/6 and C3H background and subsequently bred into a C57BL/6 background. The <italic>TgA</italic> and <italic>TgD</italic> alleles were maintained on a pure FVB/N background with one exception: the <italic>TgA</italic> mice that were used in the cross with <italic>C201R</italic> mice were on a FVB/N X C57BL/6 F1 hybrid background. Therefore, experiments were done primarily in a [FVB/N X BL/6] F1 background unless two matings were required to generate homozygotes or compound heterozygotes, in which case mice were effectively in an [N2] C57BL/6 background. All control animals were siblings of the experimental genotypes.</p><p>Spinal cord and sciatic nerve were isolated from animals immediately after they were euthanized by CO<sub>2</sub> inhalation. The tissues were frozen in liquid nitrogen and stored at &#8722;80&#176;C. The tissues were then homogenized in 1% NP-40 in phosphate buffered saline (PBS) supplemented with Protease Inhibitor Cocktail Tablets (Roche, Basal, Switzerland) using a PowerGen Model 125 Homogenizer (Fisher Scientific, Pittsburgh, PA) then centrifuged at 14,000 g for 10 min at 4&#176;C. Cleared homogenates were then sonicated at 4&#176;C and centrifuged again at 14,000 g for 10 min. Protein concentrations were assessed using a Bradford assay (BioRad, Hurcules, CA). 20 &#181;g of protein was then analyzed by immunoblot.</p><p>Protein lysates were resolved on Novex 10% Tris-Glycine Gels (Invitrogen, Carlsbad, CA) and transferred to an Invitrolon PVDF membrane for western blot analysis. Membranes were blocked with 5% skim milk in TBST (1&#215; Tris-buffered saline, 0.1% Tween-20), and incubated overnight with GARS rabbit polyclonal antibody ab42905 (1&#8758;2,000) (Abcam, Cambridge, MA) and &#946; actin mouse monoclonal antibody clone AC-74 (1&#8758;10,000)(Sigma Aldrich, St. Louis, MO) diluted in blocking solution at 4&#176; C. Following three 10 min washes in TBST, the blots were incubated with the appropriate horseradish peroxidase-conjugated secondary antibodies (Jackson ImmunoResearch, West Grove, PA) diluted in blocking solution. After three 10-min washes in TBST, the blots were developed using Western Lightening Plus-ECL, Enhanced Chemiluminescence Substrate (Perkin Elmer, Waltham, MA).</p><p>Teased nerve fibers were stained as described in the methods of <xref ref-type="bibr" rid="pgen.1002399-Stum1">[21]</xref>, <xref ref-type="bibr" rid="pgen.1002399-Arroyo1">[41]</xref>. In brief, sciatic nerves were excised, immediately placed in fresh 4% paraformaldehyde, and fixed on ice for 15 minutes. Nerves were then transferred to ice cold PBS for dissection and teasing. Using #5 forceps and 30 gauge needles, nerve sheaths were removed and the nerves were cut into 1 cm segments and teased apart at one end. The nerve segments were then transferred to a fresh Superfrost Plus Gold slides (Fisher Scientific, Pittsburgh, PA) and pulled from a drop of PBS onto a dry section of the slide to straighten the fibers for imaging. The slides were then dried overnight at room temperature and incubated in acetone at &#8722;20&#176;C for 10 min. The samples were then rehydrated with two 5 min incubations in PBS and blocked with 5% normal goat serum in PBS with 0.5% Triton X-100 for 1 h at room temperature. Primary antibodies rabbit anti-GARS (1&#8758;500) (Abcam), mouse anti-neurofilament with the 2H3 antibody (1&#8758;500)(Developmental Studies Hybridoma Bank, Iowa City, IA) were gently placed on the slide, covered with Parafilm coverslips (Pechinery Plastic, Chicago, IL) and stored in a humidified chamber overnight at 4&#176; Celsius. After three 5-min washes in PBS, the samples were incubated in the following secondary antibodies diluted 1&#8758;1,000 in blocking solution: AlexaFluor 555 goat anti-rabbit, and AlexaFluor 488 goat anti-mouse IgG<sub>1</sub> (y1) (Invitrogen, Carlsbad, CA). The samples were covered with Parafilm coverslips and incubated for 2 h at room temperature. The samples were then washed three times in PBS for 5 min each in watch glasses and mounted with Vectashield (Vector Labs, Burlingame, CA).</p><p>Aminoacylation assays were performed at room temperature in a reaction mixture containing 50 mM HEPES (pH 7.5), 20 mM KCl, 2 mM ATP, 5 mM MgCl<sub>2</sub>, 1 mM DTT, 19 &#181;M L-glycine (or L-alanine), 1 &#181;M <sup>3</sup>H-L-glycine (or <sup>3</sup>H-L-alanine), 2 &#181;M transcribed human tRNA<sup>Gly</sup>
<sub>CCC</sub> [for GlyRS activity] or 120 &#181;M yeast total tRNA (Roche Diagnostics, Indianapolis, IN) [for AlaRS activity] and 30 &#181;g total protein from tissue homogenates. Reactions were initiated by addition of reaction mixture and tRNA to tissue homogenates. Aliquots were quenched at different time points and precipitated in 96-well Multiscreen filter plates (Millipore) as described previously <xref ref-type="bibr" rid="pgen.1002399-Beebe1">[42]</xref>. After washing and elution by NaOH, samples were counted in a MicroBeta plate reader (PerkinElmer Life Sciences). Initial rates were measured from slopes obtained by fitting the data to a non-linear regression curve using GraphPad Prism 5 software. For TgA, 3 wild type and 4 transgenic littermates were sampled at 9 weeks of age: for TgD, 4 wild type and 3 transgenic littermates were tested at 6 weeks of age.</p><p>The sensory and motor branches of the femoral nerve were isolated and fixed overnight in 2% glutaraldehyde and 2% paraformaldehyde in a 0.1 M cacodylate buffer. The tissue was then processed for transmission electron microscopy and embedded in plastic before 0.5 &#181;m sections were cut and stained with toluidine blue. For more information see <xref ref-type="bibr" rid="pgen.1002399-Burgess1">[43]</xref>. For axon counting and axon diameter measurement the images were captured using a Nikon Eclipse E600 microscope with 40&#215; and 100&#215; objectives. Axon counts were done using the Cell Counter Plug-in in ImageJ. Left and right nerves were averaged. Axon diameters were measured using the Measure and Label Plugin, also in ImageJ.</p><p>Mouse plantaris muscles were surgically removed and fixed in freshly prepared 2% paraformaldehyde in PBS for four hours. The samples were then transferred to a blocking and permeabilizing solution of 5% normal goat serum and 0.5% Triton-X 100 in PBS for 1 h before they were pressed between two glass slides using a binder clip for 15 min, after which they were returned to the blocking and permeabilizing solution. The samples were then incubated overnight at 4&#176;C with 1&#8758;1,000 dilutions of anti-SV2 and anti-neurofilament (2H3) primary antibodies (Developmental Studies Hybridoma Bank, Iowa City, IA). After at least three 1 h washes in PBS with 0.5% Triton-X 100, the samples were transferred to blocking and permeabilizing solution with AlexaFluor 488 goat anti-mouse IgG<sub>1</sub> (y1) (Invitrogen, Carlsbad, CA) and &#945;-bungarotoxin conjugated with Alexa Fluor 594. After incubation overnight at 4&#176;C, the samples were washed three times for 1 h each and mounted with Vectashield mounting media (Vector Labs, Burlingame, CA) and imaged using a confocal microscope.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Confocal images were gathered using a Carl Zeiss LSM 710 or Leica SP5 laser-scanning confocal microscope with a 63&#215; objective. Z stacks were collapsed into projected images and merged using ImageJ (NIH, <ext-link ext-link-type="uri" ns0:href="http://rsb.info.nih.gov/ij/">http://rsb.info.nih.gov/ij/</ext-link>). The color balance of the NMJ images was adjusted for clarity.</p><p>Sciatic nerve conduction velocity was calculated by measuring the latency of compound motor action potentials recorded in the muscle of the left rear paw. The mice were anesthetized with 1% isofluorane and placed on a thermostatically regulated heating pad to maintain normal body temperature. Action potentials were produced by subcutaneous stimulation at the sciatic notch and at the ankle. For recording, the active needle electrode was inserted in the center of the paw and a reference electrode was placed in the skin between the first and second digits.</p><p>Statistical tests were performed using GraphPad's Prism 5 software. Statistical significance was determined using a one-way ANOVA and a post hoc Tukey test for individual differences when appropriate. A threshold of p&lt;0.05 was considered significant. The use of other tests is noted in the text. All results are presented as means &#177; SD.</p>