<p>Informed consent was obtained for all subjects in all phases of this study, with input from local communities. These studies were approved by local institutional review boards and the relevant offices at each institution contributing samples (detailed information on approvals and consents for all samples available in <xref ref-type="supplementary-material" rid="pgen.1002554.s004">Text S1</xref>).</p><p>Subjects representing the three main continental ancestral groups making up modern Latin American populations were obtained from a variety of sources. Hapmap Phase III genotype data for African and European populations was downloaded for this project, including West African (Yoruba in Ibadan, Nigeria, YRI) and East African (Luhya in Webuye, Kenya, LWK) as well as Northern European (Utah residents with ancestry from Northern and Western Europe, CEU) and Southern European (Toscani in Italy, TSI) individuals <xref ref-type="bibr" rid="pgen.1002554-Frazer1">[47]</xref>, <xref ref-type="bibr" rid="pgen.1002554-Altshuler1">[48]</xref>. For populations including parent/child trios or duos (CEU, YRI), only genotypes from the parents were used. In addition, known cryptically related individuals were removed <xref ref-type="bibr" rid="pgen.1002554-Pemberton1">[49]</xref>. Genotyping data for Europeans was further supplemented by a cohort of 619 samples of Spanish individuals, genotyped on the Affymetrix SNP 6.0 platform.</p><p>One hundred and thirty-one Native American subjects, from Mesoamerica (Nahua from Central Mexico, n&#8202;=&#8202;14, Zapotecas from Oaxaca, Mexico, n&#8202;=&#8202;21, and Maya from Campeche, n&#8202;=&#8202;25) <xref ref-type="bibr" rid="pgen.1002554-Bonilla1">[16]</xref>, <xref ref-type="bibr" rid="pgen.1002554-SilvaZolezzi1">[30]</xref>, <xref ref-type="bibr" rid="pgen.1002554-INMEGEN1">[50]</xref>, from the Sierra Madre Occidental region (Tepehuanos from Durango in Northern Mexico, n&#8202;=&#8202;22) and South America (Aymara from La Paz, Bolivia, n&#8202;=&#8202;25, and Quechua from Cerro de Pasco, Peru, n&#8202;=&#8202;24) <xref ref-type="bibr" rid="pgen.1002554-Brutsaert1">[14]</xref>, <xref ref-type="bibr" rid="pgen.1002554-Vargas1">[51]</xref> were used to determine Native American allele frequencies. These populations were genotyped either on the Affymetrix SNP 6.0 or on two platforms, the Affymetrix SNP 500K and the Illumina 550.</p><p>A summary of the populations used for this study and genotyping platforms is given in <xref ref-type="table" rid="pgen-1002554-t005">Table 5</xref>. Although, additional Native American subjects with genomewide data are available from the Human Genome Diversity Panel, these subjects were genotyped on the Illumina HumanHap 650k, and the intersection with the genotyping platforms used in our samples would have left fewer markers to be evaluated.</p><p>Four major quality control tests were performed on the data using the program plink <xref ref-type="bibr" rid="pgen.1002554-Purcell1">[52]</xref>. Individuals were excluded if they had greater than 10% missing alleles, if they were known to be related, or showed cryptic relatedness. For Native American populations, pairwise individuals were considered to have cryptic relatedness if their IBS scores showed a Z1&gt;0.15 or a Z2&gt;0.03 or if they had a proportion IBD (pi hat)&gt;0.08 <xref ref-type="bibr" rid="pgen.1002554-Weir1">[53]</xref>. Europeans and African individuals were considered cryptically related if they had a Z1&gt;0.03 or Z2&gt;0.03, or if they had a proportion IBD&gt;0.03. SNPs were included if the genotyping rate was greater than 90% and excluded if they failed a &#967;<sup>2</sup> test for Hardy-Weinberg equilibrium at a significance threshold of 10<sup>&#8722;5</sup>.</p><p>Markers representing the intersection of the genotyping platforms used to genotype the ancestral populations, which met quality control criteria (n&#8202;=&#8202;319,665) were used as a basis for selecting AIMs. <xref ref-type="fig" rid="pgen-1002554-g005">Figure 5</xref> summarizes the methodology used to select AIMs. For each SNP for each ancestral group, allele frequency was calculated with the program plink <xref ref-type="bibr" rid="pgen.1002554-Purcell1">[52]</xref>. For each marker, statistics of informativeness, including delta, <italic>F<sub>st</sub></italic>
<xref ref-type="bibr" rid="pgen.1002554-Holsinger1">[54]</xref>, and Rosenberg's informativeness for assignment statistic <italic>I<sub>n</sub></italic>
<xref ref-type="bibr" rid="pgen.1002554-Rosenberg1">[55]</xref> were calculated between each pair of ancestral populations (African/European, European/Native American, and African/Native American) based on reference allele frequencies. Locus specific branch length (LSBL) <xref ref-type="bibr" rid="pgen.1002554-Shriver1">[56]</xref> statistics were created for each population and each statistic of informativeness to translate the pairwise metrics into a population-specific statistic. A balanced set of AIMs was selected by ensuring that the cumulative LSBL for each population was approximately equal. At each stage, we selected the polymorphism with the highest LSBL for the population with the lowest cumulative LSBL that met the inclusion criteria. Polymorphisms were excluded if they were in linkage disequilibrium (r<sup>2</sup>&#8805;0.1) or within a predefined physical distance (&#8804;500 kb pairs) of previously selected AIMs. This ensures maximum independent informativeness and that the AIMs were well distributed throughout the genome. In addition, in order for potential AIMs to be applicable to all subpopulations within a continental group, potential AIMs were also excluded if there was evidence of significant allele frequency heterogeneity between the samples representing each ancestral group (&#967;<sup>2</sup> p-value&lt;0.01). A script in the Python programming language that implements this algorithm and ancestral population allele frequency data are available for download.</p><p>In order to validate the panel of AIMs, estimates of ancestry using the panel were compared to estimates of ancestry using genome-wide data. Four admixed samples were used for validation. The first two datasets were parents of Puerto Rican and Mexican subjects with asthma genotyped on the Affymetrix 6.0 GeneChip as part of the Genetics of Asthma in Latino Americans (GALA) study <xref ref-type="bibr" rid="pgen.1002554-Burchard1">[20]</xref>. The third sample consists of 1,310 individuals from Mexico City participating in a type 2 diabetes study that were genotyped with the Affymetrix 5.0 GeneChip. The fourth sample contains 312 subjects in the Mexican Genome Diversity Project (MGDP) recruited by the National Institute of Genomic Medicine (INMEGEN) from throughout Mexico, including 48 subjects from Guanajuato, 50 subjects from Guerrero, 48 subjects from Sonora, 17 subjects from Tamaulipas, 50 subjects from Veracruz, 49 subjects from Yucatan, and 50 subjects from Zacatecas <xref ref-type="bibr" rid="pgen.1002554-SilvaZolezzi1">[30]</xref>, <xref ref-type="bibr" rid="pgen.1002554-INMEGEN1">[50]</xref>. A map of the geographic distribution of MGDP-INMEGEN samples is shown in <xref ref-type="supplementary-material" rid="pgen.1002554.s002">Figure S2</xref>. A description of all the validation samples is shown in <xref ref-type="table" rid="pgen-1002554-t006">Table 6</xref>.</p><p>We implemented a three-population model to estimate individual ancestry proportions from genome-wide data using the program ADMIXTURE <xref ref-type="bibr" rid="pgen.1002554-Alexander1">[57]</xref>. We filtered our genome-wide markers to eliminate markers in linkage disequilibrium at r<sup>2</sup>&gt;0.8. Genotypes from ancestral populations described above defined the ancestral clusters relevant to Latin Americans. We also estimated ancestry using the panel of AIMs identified with the protocol above. The performance of the AIMs panel was established by calculating the correlation coefficient (R<sup>2</sup>) and measures of discordance (mean error, mean absolute error, and root mean squared error).</p><p>Using the validated AIMs panel, we genotyped 18 populations collected from Bolivia, Colombia, Venezuela, Argentina, and Chile. A description of the origin of the samples is provided in <xref ref-type="supplementary-material" rid="pgen.1002554.s004">Text S1</xref> and in <xref ref-type="table" rid="pgen-1002554-t007">Table 7</xref> and <xref ref-type="fig" rid="pgen-1002554-g006">Figure 6</xref>.</p><p>Labels in purple correspond to the Native American ancestral populations, labels in red to the validation samples, and labels in black to the 18 populations from throughout the Americas. MGDP-INMEGEN samples were collected throughout Mexico (see <xref ref-type="supplementary-material" rid="pgen.1002554.s001">Figure S1</xref>). GALA Mexico samples were also collected in the San Francisco Bay Area, CA. GALA Puerto Rico samples were also collected in New York, NY.</p><p>Subjects were genotyped on a Sequenom platform with the 400 most informative AIMs identified in phase I. AIMs were included in the final analysis if they had a genotyping call rate greater than 95% and Hardy-Weinberg equilibrium in each population individually. We required that all samples had genotyping missing data rates of &lt;10%. Sample population groups were excluded if they have average genotyping data rates of &lt;10%</p><p>File merging, strand flipping, allele frequency determination, linkage disequilibrium calculations, and identity by descent estimations were performed with the program plink <xref ref-type="bibr" rid="pgen.1002554-Purcell1">[52]</xref>. The algorithm to develop the panel of AIMs was implemented in Python version 2.6 <xref ref-type="bibr" rid="pgen.1002554-Rossum1">[58]</xref>. Individual ancestral estimates were performed with a three-population model using a model-based likelihood estimation using the program ADMIXTURE <xref ref-type="bibr" rid="pgen.1002554-Alexander1">[57]</xref>. Statistical analyses were performed with R and Python <xref ref-type="bibr" rid="pgen.1002554-Rossum1">[58]</xref>, <xref ref-type="bibr" rid="pgen.1002554-Team1">[59]</xref>. Estimation of time since admixture was performed using the program ADMIXMAP <xref ref-type="bibr" rid="pgen.1002554-McKeigue1">[60]</xref>, <xref ref-type="bibr" rid="pgen.1002554-Hoggart2">[61]</xref>, <xref ref-type="bibr" rid="pgen.1002554-Montana1">[62]</xref>, assuming an average of 29 years per generation <xref ref-type="bibr" rid="pgen.1002554-Fenner1">[63]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Source code for the AIMs selection script is available at <ext-link ext-link-type="uri" ns0:href="http://bts.ucsf.edu/burchard/">http://bts.ucsf.edu/burchard/</ext-link>.</p>