<p>The creation of the NAM population of RIL families has been described previously <xref ref-type="bibr" rid="pgen.1002383-Buckler1">[3]</xref>, <xref ref-type="bibr" rid="pgen.1002383-McMullen1">[28]</xref>. Environments, field design, traits. Another publicly-available maize RIL family, the intermated B73-by-Mo17 (IBM) family, was also included in our analyses for a total of 4892 RILs from 26 biparental families with B73 as a common parent, and a total of 27 parents. All inflorescence traits were measured in eight environments, including Aurora, NY, Clayton, NC, Urbana, IL, Homestead, FL, and Ponce, PR in 2006, and Aurora, NY in 2007. Tassel traits were additionally measured in Columbia, MO in 2006 and Urbana, IL in 2007. Ear traits were additionally measured in Clayton, NC in 2007 and Aurora, NY in 2008. In each location, each family was represented by 220 rows: 200 rows of RILs and 10 rows of each parent. Data from some RILs was later discarded to bring the total RIL number to 4892 <xref ref-type="bibr" rid="pgen.1002383-McMullen1">[28]</xref>.</p><p>Trait transformations were performed using the boxcox function in R with lambda ranging from &#8722;10 to +10 in increments of 0.1, where lambda values of 0 and 1 are equivalent to log and linear transformations, respectively. Branch number and ear row number traits had maximum likelihood values of lambda of 0.3 and 0.4 respectively. Box-cox transformed values of these traits were used to calculate BLUPs. BLUPs were calculated in SAS using PROC MIXED and a model with location, set(location), family, family*location, and entry(family) as random effects.</p><p>The genotypic dataset consisted of 836 markers, representing the subset of 1106 markers that could be placed unambiguously on the physical map, scored on 4892 RILs. Missing data, consisting primarily of markers that were non-informative in particular families, were imputed as previously described <xref ref-type="bibr" rid="pgen.1002383-Tian1">[4]</xref>. Joint linkage models were obtained in SAS using the stepwise selection procedure in PROC GLMSELECT. The family term was forced into the model, and each of the 836 possible marker-by-family terms was made available for inclusion. Significance levels for entry and exit of model terms were determined by permutation: phenotypic data were permuted against the genotypic data separately within each family, all 836 marker-by-family terms were tested, and the lowest resulting p-value was recorded for each permutation. 1000 permutations were performed, and alpha was set at .05.</p><p>Missing SNP data from the maize HapMap project <xref ref-type="bibr" rid="pgen.1002383-Gore1">[29]</xref> were imputed as previously described <xref ref-type="bibr" rid="pgen.1002383-Tian1">[4]</xref>. For non-recombinant RIL marker intervals, SNP values of 0 (common parent allele) and 1 (alternate allele) were assigned according to the parental genotype. For recombinant RIL marker intervals, SNP values between 0 and 1 were assigned based on the physical position of the SNP within the interval and assuming a linear relationship between physical and genetic distance. Projection was also tested assuming a linear relationship between &#8220;genespace&#8221; and genetic distance, but this had very little effect on the results. GWAS models were fit for each chromosome separately. The phenotypes for each chromosome consisted of residuals from a joint linkage model excluding both the family covariate and all QTL on the chromosome under consideration. GWAS genotypes were obtained by scoring 1.6 million SNPs in the 27 parental lines and then &#8220;projecting&#8221; these genotypes into the progeny RILs. We employed a subsampling procedure wherein 80% of the RILs from each family were sampled without replacement, and forward regression was used to fit SNPs in the presence of the family term using permutation-derived significance thresholds <xref ref-type="bibr" rid="pgen.1002383-Tian1">[4]</xref>. This process was repeated 100 times to obtain a resample model inclusion probability (RMIP) value for each SNP ranging from 0 to 1, which represents the percentage of samples in which that SNP was selected. Only SNPs with RMIP values greater than or equal to 0.05 were used for further analysis.</p><p>QTL effects for each trait were divided by the standard deviation of BLUP values across a set of 282 diverse maize lines that included the 27 parental lines, and multiplied by the broad-sense heritability estimate for that trait. Since a minimum of 26 QTL were detected for each trait, a 26-QTL model was refit for each trait and used to determine effect sizes. This experiment used a reference design (26 inbred lines were each crossed to a common parent), meaning that QTL effect sizes are potentially biased for traits for which the common parent is an outlier. To circumvent this problem, for each QTL we calculated the predicted effects of all pairwise matings between the 26 parents (eg: for two parents with effects of +1 and &#8722;1 relative to the common parent, the predicted QTL effect size in this family is 2), yielding a total of 325 (26 choose 2) effect sizes for each QTL, or a total of 6825 qtl effects per trait.</p><p>Pleiotropy between pairs of traits in the joint linkage analysis was evaluated as described previously <xref ref-type="bibr" rid="pgen.1002383-Buckler1">[3]</xref>. Briefly, the QTL model for each trait was applied to every other trait, and correlations between effect estimates were used to detect significant pleiotropic QTL. For each QTL in each pairwise trait comparison, the Pearson correlation coefficient <italic>(r)</italic> between the two effect vectors of length 26 is significant at <italic>p</italic>&lt;0.01 if <italic>r</italic> exceeds 0.495 (two tailed <italic>t</italic> distribution, 24 d.f.). The percentage of shared QTL between two traits is the sum of two fractions: the fraction of significant correlations when the model for trait 1 is applied to trait 2, and vice versa. Pleiotropy between pairs of traits in GWAS analysis has not been reported previously. First, the effects of all GWAS SNPs for each trait in each family were weighted by their RMIP values and averaged in sliding windows across the genome, in order to derive a vector of effect estimates for each trait in each window. Results presented here used a 5 cM window size and a 2.5 cM step, but similar results were obtained for larger and smaller windows. Second, for each pair of traits, only windows where the sum of RMIP values for each trait fell above a threshold (RMIP&#8202;=&#8202;0.10 for the results presented) were considered. Finally, significance of Pearson correlation coefficients between effect estimates was calculated as for joint linkage analysis.</p><p>We considered only the top ten GWAS SNPs for each trait, ordered by decreasing RMIP value, on the assumption that these more robustly-selected SNPs should be more closely linked to the causal variants. To test for significant enrichment, the number of high-RMIP SNPs for a given trait that fell within 0.5, 1, and 2 cM of candidates was compared with a null distribution obtained by selecting an equivalent number of random genes (eg: 17 random genes for comparison to 17 SBP candidates), calculating their proximity to trait SNPs, and repeating this process 1000 times. Selection of random positions rather than random genes represents a far less stringent test, since genes are clustered in the maize genome.</p>