<p>The two strains of <italic>C. amylolentus</italic>, CBS6039 and CBS6273, and the one <italic>T. wingfieldii</italic> isolate, CBS7118, were obtained from the Centraalbureau voor Schimmelcultures (CBS) Fungal Biodiversity Centre in the Netherlands. Both CBS6039 and CBS6273 were originally isolated from insect frass in South Africa, while CBS7118 was originally isolated from rubber sheet in Indonesia. All species were grown and maintained on yeast extract-peptone-dextrose (YPD) medium at 24&#176;C. Mating assays were performed on V8 medium pH&#8202;=&#8202;5 in the dark and also at 24&#176;C. Random spore dissection was performed on YPD medium as previously described <xref ref-type="bibr" rid="pgen.1002528-Idnurm1">[44]</xref>. Spore chain dissection was performed by first transferring a well separated spore chain onto a drop of zymolyase on YPD, and after incubation at 24&#176;C for 15 minutes, individual spore from the spore chain was dissected as previously described <xref ref-type="bibr" rid="pgen.1002528-Idnurm1">[44]</xref>.</p><p>To isolate genomic DNA from <italic>T. wingfieldii</italic> and <italic>C. amylolentus</italic>, cells were cultured in 50 ml of liquid YPD shaking overnight at 24&#176;C. The pellets were then lyophilized overnight and the CTAB method of fungal DNA isolation was performed as described before <xref ref-type="bibr" rid="pgen.1002528-Metin1">[24]</xref>. Plasmid DNA from positive TOPO clones was extracted using the QIAprep Spin Miniprep Kit (Qiagen, Valencia, CA), fosmid DNA was isolated using a modified miniprep protocol, and DNA from the shot-gun sequencing libraries was extracted using the DirectPrep96 Miniprep Kit (Qiagen, Valencia, CA). Additionally, progeny DNA was isolated using a modified miniprep protocol and colony lifts were performed to isolate DNA from individual colonies in each fosmid library according to the protocol described in <xref ref-type="bibr" rid="pgen.1002528-RodriguezCarres1">[28]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We designed degenerate PCR primers using the online computer program, COnsensus-DEgenerate Hybrid Oligonucleotide Primer (CODEHOP, <ext-link ext-link-type="uri" ns0:href="http://blocks.fhcrc.org/codehop.html">http://blocks.fhcrc.org/codehop.html</ext-link>) to identify <italic>MAT</italic> specific genes in <italic>T. wingfieldii</italic>. The primers consist of a relatively short 3&#8242; degenerate core and a longer 5&#8242; non-degenerate consensus clamp designed by multiple sequence alignments <xref ref-type="bibr" rid="pgen.1002528-Rose1">[48]</xref>. We aligned sequences for two flanking genes (<italic>FAO1</italic> and <italic>NOG2</italic>) and two recently acquired <italic>MAT</italic> genes (<italic>RPO41</italic> and <italic>LPD1</italic>) from <italic>C. neoformans</italic> var. <italic>neoformans</italic> and var. <italic>grubii</italic>, <italic>C. gattii</italic>, <italic>U. maydis</italic>, and <italic>C. cinerea</italic> to design the degenerate PCR primers (see <xref ref-type="supplementary-material" rid="pgen.1002528.s017">Table S4</xref> for primer information). PCR was performed on genomic DNA isolated by the CTAB extraction method as template and products were separated by gel electrophoresis. Products with the strongest ethidium bromide-staining signal were then gel extracted using the QIAquick Gel Extraction Kit (Qiagen, Valencia, CA) followed by transformation into <italic>E. coli</italic> using the TOPO-TA cloning Kit (Invitrogen, Carlsbad, CA). Plasmid DNA was purified from transformants and then sequenced. For <italic>C. amylolentus</italic>, degenerate primers were not used. Instead, primers from <italic>T. wingfieldii</italic> were directly used to amplify <italic>MYO2</italic>, <italic>LPD1</italic>, <italic>SXI1</italic>, and <italic>SXI2</italic> in both <italic>C. amylolentus</italic> strains (see <xref ref-type="supplementary-material" rid="pgen.1002528.s017">Table S4</xref> for primer information).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We employed the CopyControl Fosmid Library Production Kit (Epicentre, Madison, WI) to generate fosmid libraries for <italic>T. wingfieldii</italic> and <italic>C. amylolentus</italic> strain CBS6039. At least 2.5 &#181;g of CTAB isolated genomic DNA was randomly sheared using a 200 &#181;l small bore pipette tip and sheared DNA was end-repair converted to blunt 5&#8242; phosphorylated ends using End-Repair Enzyme Mix, dNTPs, and ATP. We then separated the end-repaired DNA overnight using a Contour-clamped Homogenous Electric Field (CHEF) on a CHEF DR-II apparatus (Bio-Rad, Hercules, CA). The following conditions were used: 1- to 6-second switch time, 6 V/cm, 14&#176;C for 14&#8211;15 hrs in 0.5X TBE. The size-fractionated DNA, 25 to 40 kb fragments, was recovered by gel extraction and the DNA was precipitated with sodium acetate and ethanol. The precipitated insert DNA was then ligated into the CopyControl pCC1FOS cloning-ready vector and incubated overnight at 24&#176;C. The ligated DNA was packaged in phage particles and plated on <italic>E. coli</italic> phage-resistant cells (EPI100-T1R plating strain) overnight at 37&#176;C (detailed protocol can found at <ext-link ext-link-type="uri" ns0:href="http://www.epibio.com/item.asp?ID=385">http://www.epibio.com/item.asp?ID=385</ext-link>). Approximately 16,000 fosmid clones were picked into 96-well plates and transferred to 384-well plates for long-term storage at &#8722;80&#176;C. The 384-well plates were replicated onto high-density filters for hybridizations using the <italic>MAT</italic> genes.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Positive fosmid clones were sequenced using the shot-gun sequencing method described by Metin et al. <xref ref-type="bibr" rid="pgen.1002528-Metin1">[24]</xref>. Six fosmids were pooled and sequenced to generate the assembly for <italic>T. wingfieldii</italic> and four fosmids were individually sequenced to generate the assembly for <italic>C. amylolentus</italic> strain CBS6039. Sequencing reactions were performed using Big Dye chemistry v3.1 (Applied Biosystems, Foster City, California, United States) and analyzed on an Applied Biosystems 3730xl capillary sequencer in the Biological Sciences Sequencing Facility at Duke University. For each library, approximately &#8764;1200 sequence reads were imported into UNIX using Phred and Phrap to assemble the sequences into larger contigs of overlapping sequence <xref ref-type="bibr" rid="pgen.1002528-Gordon1">[49]</xref>, <xref ref-type="bibr" rid="pgen.1002528-Gordon2">[50]</xref>, <xref ref-type="bibr" rid="pgen.1002528-delaBastide1">[51]</xref>. To close gaps in the assemblies, we designed primers from contig ends using Primer 3 (<ext-link ext-link-type="uri" ns0:href="http://frodo.wi.mit.edu/primer3/">http://frodo.wi.mit.edu/primer3/</ext-link>). The GenBank accession numbers for <italic>T. wingfieldii</italic> are HM368525 (HD locus) and HM368524 (P/R locus). The GenBank accession numbers for the HD locus and the three P/R contigs in <italic>C. amylolentus</italic> CBS6039 are: HM640220 (HD locus), HM640221 (<italic>RPL39-MYO2</italic>), HM640222 (<italic>LPD1-STE12</italic>), and HM640223 (<italic>GEF1-MFA</italic>). The GenBank accession numbers for genes from <italic>C. amylolentus</italic> CBS6273 are: HM640224 (<italic>SXI1</italic>), HM640225 (<italic>SXI2</italic>), HM640226 (<italic>GEF1</italic>), HM640227 (<italic>LPD1</italic>), and HM640228 (<italic>ETF1</italic>).</p><p>To determine the ploidy of the two <italic>C. amylolentus</italic> and one <italic>T. wingfieldii</italic> strains, we cultured the isolates on YPD medium for 2 days at 24&#176;C. Each isolate was processed for flow cytometry as previously described <xref ref-type="bibr" rid="pgen.1002528-Lin1">[5]</xref>, <xref ref-type="bibr" rid="pgen.1002528-Tanaka1">[52]</xref> and analyzed using the FL1 channel on a Becton-Dickinson FACScan. The &#8764;20 Mb genome of <italic>C. neoformans/gattii</italic> was used as a reference for ploidy determination (including haploid and diploid controls).</p><p>To isolate chromosomal DNA of <italic>C. amylolentus</italic> and <italic>T. wingfieldii</italic>, spheroplasts were generated following the spheroplasting protocol for <italic>C. neoformans</italic> and <italic>C. gattii</italic>
<xref ref-type="bibr" rid="pgen.1002528-Wickes2">[53]</xref>. The plugs containing spheroplasts were lysed at 55&#176;C for at least 24 hrs in lysing solution (0.5 M EDTA/10 mM Tris-Cl (pH&#8202;=&#8202;10) and 1% Sarcosyl) and then loaded onto a PFGE apparatus and separated for approximately 5 days on a CHEF DR-II apparatus (Bio-Rad, Hercules, CA). The following conditions were used: Block 1: 75- to 150-second switch time, 4 V/cm, 13&#176;C for 30 hrs and Block 2: 200 to 400-second switch time, 4 V/cm, 13&#176;C for 60 hrs in 0.5X TBE. The gel was then stained in ethidium bromide for 15 minutes, destained for an hour, and visualized using a UV lamp. The chromosomal DNA was blotted overnight onto Hybond (Amersham, Piscataway, NJ) membranes in 20X SSC using standard protocols. The membrane was then hybridized to <italic>MAT</italic> gene probes generated by PCR. We also performed Southern blot analysis on genomic DNA from <italic>C. amylolentus</italic> that was digested with EcoRV, PstI, BamHI, or NotI. The digested DNA was separated on an agarose gel and probed with the <italic>RPL22</italic> gene probe amplified from <italic>C. amylolentus</italic>, with primers designed for <italic>T. wingfieldii</italic> (see <xref ref-type="supplementary-material" rid="pgen.1002528.s017">Table S4</xref> for primer information).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We compared sequences from the HD locus of <italic>T. wingfieldii</italic> to those of <italic>C. amylolentus</italic> by employing a matrix comparison (or dot plot) analysis. To generate each dot plot, we employed the Molecular Toolkit's online nucleic acid dot plots program (<ext-link ext-link-type="uri" ns0:href="http://www.vivo.colostate.edu/molkit/dnadot/">http://www.vivo.colostate.edu/molkit/dnadot/</ext-link>). The parameters for the dot plot analyses were as follows: the window size was 51 and the mismatch limit was 6. We also employed the bioinformatic software, Artemis Comparison Tool Release 8 (<ext-link ext-link-type="uri" ns0:href="http://www.sanger.ac.uk/resources/software/">http://www.sanger.ac.uk/resources/software/</ext-link>) to generate comparison plots across <italic>MAT</italic> of <italic>T. wingfieldii</italic> to <italic>C. amylolentus</italic> and both sibling species compared to <italic>C. neoformans</italic> serotype D strain JEC21 <xref ref-type="bibr" rid="pgen.1002528-Rutherford1">[54]</xref>. The input file was created using WebACT (<ext-link ext-link-type="uri" ns0:href="http://www.webact.org/WebACT/home">http://www.webact.org/WebACT/home</ext-link>) with the Blastn algorithm <xref ref-type="bibr" rid="pgen.1002528-Carver1">[55]</xref>.</p><p>Phylogenetic analysis was performed on coding sequences using MEGA 5 <xref ref-type="bibr" rid="pgen.1002528-Tamura1">[56]</xref>. To determine the phylogenetic relationship, the Neighbor-Joining method based on the Kimura 2-parameter model was employed <xref ref-type="bibr" rid="pgen.1002528-Rzhetsky1">[57]</xref>. For statistical support, 500 replicates were performed and bootstrap values were calculated.</p><p>We performed Southern blot analysis using standard protocols on genomic DNA from <italic>C. amylolentus</italic> digested with BamHI, BglI, ClaI, EcoRI, or NcoI. The digested DNA was separated on an agarose gel and probed with the <italic>STE3</italic> gene, stripped (0.1% SDS and 0.1X SSC in boiling water, 3 times for 15 minutes each), and probed with the contig ends from the P/R assembly in <italic>C. amylolentus</italic> amplified by PCR (see <xref ref-type="supplementary-material" rid="pgen.1002528.s017">Table S4</xref> for primer information).</p><p>Standard description: <italic>Filobasidiella amylolenta</italic> Findley &amp; Heitman sp. nov.</p><p>Etymology: The epithet is chosen to be identical with that of <italic>C. amylolentus</italic> (Van der Walt, D.B. Scott &amp; Klift) Golubev 1981 <xref ref-type="bibr" rid="pgen.1002528-Golubev1">[58]</xref>. Heterothallic fungus. Hyphae dikaryotic, clamped connections fused. Aseptate basidia, 3&#8211;5 &#181;m diameter, terminating in four chains of basidiospores. Basidiospores are aerial, round, and 2&#8211;2.5 &#181;m in diameter.</p><p>Holotype: Mounted teleomorph is paired cultures of <italic>C. amylolentus</italic> type strain, CBS6039<sup>T</sup> (A1B1) crossed to CBS6273 (A2B2) on V8 medium (pH&#8202;=&#8202;5). These strains were originally isolated from insect frass in South Africa <xref ref-type="bibr" rid="pgen.1002528-Golubev1">[58]</xref>. A slide preparation of mating structures, basidia and basidiospores, is deposited in the USDA's Systematic Mycology and Microbiology Laboratory in Beltsville, Maryland (deposit number: BPI 881008). Strains CBS6039 (mating-type A1B1) and CBS6273 (mating-type A2B2) should be designated as the ex-type strain and the isotype strain, respectively, for the teleomorph <italic>Filobasidiella amylolenta</italic>.</p><p>Latin description: <italic>Filobasidiella amylolenta</italic> Findley &amp; Heitman sp. nov.</p><p>Fungus heterothallicus. Hyphae dikaryoticae, fibulis fusis. Basidia aseptata, 3&#8211;5 &#181;m lata, quatuor catenas basidiosporarum producentia. Basidiosporae aeriae, globosae, 2&#8211;2.5 &#181;m diametro.</p><p>Spores and yeast cells were cultured on slides coated with V8 pH&#8202;=&#8202;5 medium for one week or longer to allow production of mating structures. The slide was first washed with phosphate buffered saline (PBS) followed by staining the cell wall using a solution of Calcofluor white (fluorescent brightener 28 F-3397; Sigma) for 15 minutes. Slides were rinsed with PBS and fixed for 15 minutes in fixing solution (3.7% formaldehyde and 1% Triton-X100 in PBS). After permeabilization of the fungal cells, nuclear content was examined by staining with Sytox green (Molecular Probes) for 30 minutes. Slides were washed with PBS and a cover slip was applied to the slide for observation. In addition to staining spores and yeasts, mating filaments were also stained. Agar pieces were removed from mating plates and washed several times with PBS. Calcofluor white was added directly to the agar piece for 30 minutes, followed by washing with PBS, and fixing for 45 minutes. After permeabilizing samples, filaments were washed with PBS and stained with 1 mg/ml Hoechst 33258 (Invitrogen, Carlsbad, CA) overnight at 4&#176;C. The next day, samples were washed with PBS, a thin slice of the agar (containing the mating filaments) was removed using a razor blade and a mounting solution containing anti-fade (Invitrogen, Carlsbad, CA) was added to the agar slice on a slide. The slides were sealed with nail polish and stored at 4&#176;C in the dark after microscopic evaluation. All staining was performed at 24&#176;C, unless otherwise noted. SEM was performed on <italic>C. amylolentus</italic> matings incubated on V8 pH&#8202;=&#8202;5 medium for 2 weeks. The specimen was prepared and analyzed as described in <xref ref-type="bibr" rid="pgen.1002528-RodriguezCarres1">[28]</xref>. Microscopy was performed with an Axioskop 2 plus upright microscope (Zeiss). Images were captured using an AxioCam MRm camera. Scanning electron microscopy was performed and viewed on a JEOL JSM 5900LV (JEOL U.S.A., Peabody, MA) SEM at 15 kV.</p><p>Microdissection of spores (random or individual spore chains using zymolyase (Zymo Research Corp., Orange, CA, USA)) was performed on YPD medium incubated at 24&#176;C for two days to allow spores to germinate.</p><p>Genotyping of the <italic>MAT</italic> loci was achieved using a set of PCR markers (<italic>RPL39</italic>, <italic>GEF1</italic>, and <italic>STE3</italic>) and PCR-RFLP markers (<italic>SXI1</italic> (enzyme EcoRV), <italic>SXI2</italic> (enzyme RsaI), and <italic>ETF1</italic> (enzyme DdeI)). To genotype other genomic regions, we used a set of 20 RAPD markers (<xref ref-type="supplementary-material" rid="pgen.1002528.s017">Table S4</xref>). Linkage analyses indicated 18 of these 20 markers are not derived from the <italic>C. amylolentus MAT</italic> loci, with exception of markers Pi_Random_24_No.2 and JOHE22656_No.1, which were positioned in the same linkage group with HD markers. Non-<italic>MAT</italic>-association of nine of these 18 markers were further confirmed by cloning and sequencing of the polymorphic bands, as none of them was <italic>MAT</italic> specific sequence (data not shown). We designate the CBS6039 parent as A1B1 and the CBS6273 parent as A2B2, according to the designation used for a tetrapolar mating system and our findings, assigning <italic>A</italic> as the P/R locus and <italic>B</italic> as the HD locus (as in <italic>T. mesenterica</italic>, <italic>C. heveanensis</italic>, and <italic>U. maydis</italic>
<xref ref-type="bibr" rid="pgen.1002528-Metin1">[24]</xref>, <xref ref-type="bibr" rid="pgen.1002528-Bandoni1">[26]</xref>). Recombination was scored according to marker exchange for the P/R and/or HD locus. Recombination frequency among RAPD markers was inferred using program MapMaker.</p><p>These genotyping data was further analyzed using program MapMaker to generate genetic linkage groups. Additionally, the UPGMA clustering method implemented in the software MEGA 5 was used to analyze the genetic relationships among F1S2 progeny isolated from the same basidium.</p>