<p>All individuals studied and all analyses on their samples were done according to the Declaration of Helsinki and informed consents were approved by the local ethics committee for the Istituto di Ricerca Genetica e Biomedica-CNR (IRGB-CNR; Cagliari, Italy) and by MedStar Research Institute, responsible for intramural research at the National Institutes of Aging, Baltimore, Maryland, United States.</p><p>We recruited and phenotyped 6,148 individuals, males and females, ages 14&#8211;102 y, from a cluster of four towns in the Lanusei Valley of Sardinia <xref ref-type="bibr" rid="pgen.1002480-Pilia1">[11]</xref>. During physical examination, a blood sample was collected from each individual and divided into two aliquots. One aliquot was used for DNA extraction and the other to characterize several blood phenotypes, including evaluation of serum levels of hsCRP, IL-6, MCP-1, and values of ESR. Descriptive statistics of the study cohort are shown in <xref ref-type="supplementary-material" rid="pgen.1002480.s005">Table S2</xref>. Serum levels of hsCRP were measured by the high sensitivity Vermont assay (University of Vermont, Burlington), an enzyme-linked immunosorbent assay calibrated with WHO Reference Material <xref ref-type="bibr" rid="pgen.1002480-Macy1">[55]</xref>. The lower detection limit of this assay is 0.007 mg/l, with an inter-assay coefficient of variation of 5.14%. Serum levels of IL-6 and MCP-1 were measured by Quantikine High Sensitive Human Immunoassays (R&amp;D Systems, Inc.), according to manufacturer's instructions. This method employs solid-phase ELISA techniques. For IL-6, the lower detection limit is 0.039 pg/ml. The intra-assay coefficient of variations (CVs) were 6.9% to 7.8% over the range 0.43&#8211;5.53 pg/ml. For MCP-1, the lower detection limit is 5.0 pg/ml. The intra-assay coefficient of variations (CVs) were 4.7% to 7.8% over the range 76.7&#8211;1121 pg/ml. ESR was measured using sedimentation measurement tubes buffered with 3.8% sodium citrate (Venoject-Terumo). After mixing of 2.4 ml of blood with the additive, tubes were left in a vertical position in the specific support with graduation markings for 30 minutes to allow sedimentation of the erythrocytes by gravity. The erythrocyte sedimentation rate is calculated in Westergreen units (mm/h) determining the length at the plasma/erythrocyte cell interface level within the sedimentation tube. Samples affected by multiple sclerosis (MS) and type 1 diabetes (T1D) used for the side case-control analysis briefly reported in the Discussion were recruited from all the island as previously described (51). Only 20 of these samples overlapped with those in the SardiNIA cohort.</p><p>During the study, we genotyped 4,694 individuals selected from the whole sample to represent the largest available families, regardless of their phenotypic values. Specifically, 1,412 were genotyped with the 500 K Affymetrix Mapping Array set, 3,329 with the 10 K Mapping Array set, with 436 individuals genotyped with both arrays. We also recently typed 1,097 individuals with the Affymetrix 6.0 chip, of which 1,004 and 66 were also typed with the 10 K and 500 K chips respectively. This genotyping strategy allowed us to examine the majority of our cohort in a cost-effective manner since genotypes for the SNPs that passed quality control checks could be propagated through the pedigree using imputation. Measurements of inflammatory biomarkers were available for 4,137, 4,292, 4,295 and 3,596 individuals for hsCRP, IL-6, MCP-1 and ESR, respectively, among the 4,694 genotyped. A total of 731,209 autosomal SNPs passed stringent quality control checks. Quality checks for the 10 K and 500 K chips were described previously <xref ref-type="bibr" rid="pgen.1002480-Scuteri1">[56]</xref>. For the Affymetrix 6.0 chip, similar criteria were used, as detailed in <xref ref-type="supplementary-material" rid="pgen.1002480.s006">Table S3</xref>. In addition, we also removed SNPs in common between the other chips that showed an high level of discordance or that generated too many discrepancies when comparing genotypes across 11 duplicates. After performing quality control checks and merging genotypes from the three gene chip platforms, we used the quality controlled 731,209 autosomal markers to estimate genotypes for all polymorphic SNPs in the CEU HapMap population (release 22) <xref ref-type="bibr" rid="pgen.1002480-The1">[57]</xref>, in the individuals genotyped with the 500 K Array and the 6.0 Affymetrix chip separately using the MaCH software <xref ref-type="bibr" rid="pgen.1002480-Li1">[58]</xref>. Taking advantage of the relatedness among individuals in the SardiNIA sample, we carried out a second round of computational analysis to impute genotypes at all SNPs in the individuals who were genotyped only with the Affymetrix Mapping 10 K Array, being mostly offspring and siblings of the individuals genotyped at high density. At this second round of imputation, we focused on the SNPs for which the imputation procedure predicted r<sup>2</sup>&gt;0.30 between true and imputed genotypes and for which the inferred genotype did not generate an excess of Mendelian errors. We then used a modified version of the Lander-Green algorithm, as previously described <xref ref-type="bibr" rid="pgen.1002480-Willer1">[3]</xref>, <xref ref-type="bibr" rid="pgen.1002480-Scuteri1">[56]</xref> to estimate IBD sharing at the location of the SNPs being tested and identify stretches of haplotype shared with close relatives who were genotyped at higher density and probabilistically infer missing genotypes. The within-family imputation procedure and the association test are implemented in Merlin software <xref ref-type="bibr" rid="pgen.1002480-Abecasis1">[59]</xref>, <xref ref-type="bibr" rid="pgen.1002480-Chen1">[60]</xref>. Due to computational constraints, we divided large pedigrees into sub-units with &#8220;bit-complexity&#8221; of 21 or less (typically, 25&#8211;30 individuals) before analysis.</p><p>For association, we evaluated the additive effect of genotyped and imputed SNPs on inflammatory biomarker levels using a family-based association test implemented in Merlin (&#8211;fastassoc option) <xref ref-type="bibr" rid="pgen.1002480-Abecasis1">[59]</xref>, <xref ref-type="bibr" rid="pgen.1002480-Chen1">[60]</xref>. This test accounts for relatedness under the assumption that the samples analyzed are from an ethnically homogeneous population <xref ref-type="bibr" rid="pgen.1002480-Abecasis1">[59]</xref>, <xref ref-type="bibr" rid="pgen.1002480-Chen1">[60]</xref>, and this is suggested by demographic records indicating that 89% of the participants were born in the same 31 Km<sup>2</sup> area and for 95% of the volunteers, both parents and all grandparents were born in Sardinia <xref ref-type="bibr" rid="pgen.1002480-Pilia1">[11]</xref>. At each SNP, levels of each biomarker of inflammation were regressed onto allele counts in a regression model that included gender, age, and age-squared as covariates. We also used a second model, which included body-mass index (BMI) and smoking status as additional covariates, as these have been previously implicated as being associated with inflammatory biomarker levels <xref ref-type="bibr" rid="pgen.1002480-Reiner1">[24]</xref>. Here we report the results from the second model, where the inclusion of the additional covariates improved the variance explained by the model (from 1.8% to 3.7% for CRP, from 18.1% to 19.5% for ESR, from 6.8% to 7.6% for IL-6, and from 3% to 3.3% for MCP-1). Genomic control parameters show negligible inflation (1.049, 1.039, 1.031 and 1.115, respectively for hsCRP, MCP-1, IL-6 and ESR); nevertheless the corresponding correction factors were applied to the GWAS results to completely avoid spurious associations.</p><p>The SardiNIA stage 2 cohort was used to follow-up initial findings <xref ref-type="bibr" rid="pgen.1002480-ArnaudLopez1">[27]</xref>. Genotyping of specific SNPs was performed in Sardinian individuals selected for replication efforts using TaqMan single SNP genotyping assays (Applied Biosystems). In particular, we genotyped and analysed 1,392 individuals from the SardiNIA stage 2 cohort, who were unrelated (kinship coefficient&#8202;=&#8202;0) to the individuals analysed in the GWAS.</p><p>We successfully genotyped 6,145 samples using the MetaboChip and ImmunoChip arrays (Illumina). The MetaboChip was designed in collaboration with several international consortia <xref ref-type="bibr" rid="pgen.1002480-Willer1">[3]</xref>, <xref ref-type="bibr" rid="pgen.1002480-Prokopenko1">[61]</xref>, <xref ref-type="bibr" rid="pgen.1002480-Preuss1">[62]</xref> with the aim to fine map association loci detected through GWAS for a variety of traits. Part of the design included a set of wild-card SNPs chosen by individual research groups; the SardiNIA study promoted several SNPs associated with a wide range of traits, including rs12075. The ImmunoChip is also a consortium based array, designed to fine map loci associated to 12 immunologically related human diseases, or immune-mediated disease loci, as well as a set of wild-card SNPs. The SardiNIA study had not role in the design of this array, and a full detailed description is provided elsewhere <xref ref-type="bibr" rid="pgen.1002480-Cortes1">[12]</xref>. All samples had a genotyping call rate &gt;98%, and SNP genotypes were carefully assessed though several quality control checks. In particular, we removed markers with call rate &lt;98%, with strong deviation from HWE (p&lt;10<sup>&#8722;6</sup>), that were monomorphic or leading to an excess of Mendelian errors (defined as &gt;1% of the families). A detailed breakdown of markers excluded by each filter criteria is provided on <xref ref-type="supplementary-material" rid="pgen.1002480.s007">Table S4</xref>. Since the majority of the variants included on these custom arrays are of low frequency compared to the GWAS data set (average MAF&#8202;=&#8202; 0.176, compared to 0.219 observed in GWAS), the impact of hidden population structure and imprecise modelling of relatedness due to pedigree splitting (task we performed on the GWAS data set due to computational constraints) could be problematic. Analysis was thus carried out using EMMAX, a variant component model that overcomes such issues by using a genomic-based kinship matrix <xref ref-type="bibr" rid="pgen.1002480-Kang1">[63]</xref>. To calculate the kinship matrix, we used all SNPs that passed quality control checks but excluding those with MAF between 0 and 1%. Association analysis was subsequently performed testing all QCed SNPs (<xref ref-type="supplementary-material" rid="pgen.1002480.s007">Table S4</xref>), in spite of their minor allele frequency. Observed genomic lambda were 1.01, 0.962, 1.00 and 1.01, respectively for IL6, VES, MCP1, and hsCRP (as a note, genomic lambda using Merlin on the same data set were 1.41, 1, 1.26 and 1.14, respectively). To declare an association significant, we used a Bonferroni threshold of 0.05/293,875&#8202;=&#8202;1.7&#215;10<sup>&#8722;7</sup>.</p><p>The variance explained by the strongest associated SNPs was calculated, for each trait, as the difference of R2 adjusted observed in the full and the basic models, where the full model contains all the independent SNPs in addition to the covariates.</p><p>We performed conditional analysis at each locus by adding the top associated SNP to the already included covariates, and testing for association the remaining SNPs at the locus. A marker was declared independent only if the p-value observed in the conditional analysis reached genome-wide significance threshold (5&#215;10<sup>&#8722;08</sup> in the Step1 GWAS, and 1.7&#215;10<sup>&#8722;07</sup> in the custom-array based dataset) <xref ref-type="bibr" rid="pgen.1002480-The2">[64]</xref>.</p>