<p>
<italic>Schizo. pombe</italic> cells were grown in yeast extract (YE) medium. Standard genetic and molecular procedures were employed as described previously <xref ref-type="bibr" rid="pgen.1002407-Moreno1">[41]</xref>. To examine cell growth on plates, serial dilutions of cells were spotted on YEA (YE agar) plates, and incubated at 30&#176;C.</p><p>The <italic>cdc6<sup>+</sup></italic> and <italic>cdc20<sup>+</sup></italic> genes were amplified by PCR and cloned into pUC19. <italic>cdc6-L591F</italic> and <italic>cdc20-M630F</italic> mutant genes were constructed by PCR-meditated site-directed mutagenesis and sequenced to ensure that only the desired mutation was introduced. Both wild-type and mutant genes were introduced into <italic>Schizo. pombe</italic> at their native loci by recombination-mediated cassette exchange (RMCE) <xref ref-type="bibr" rid="pgen.1002407-Watson1">[17]</xref>.</p><p>Spontaneous mutation rates were determined by fluctuation assay as described previously <xref ref-type="bibr" rid="pgen.1002407-Foster1">[42]</xref>. Briefly, 11 independent single colonies were suspended in 5 ml YEP (YE+polypeptone) medium and grown to saturation at 30&#176;C. Cells were diluted appropriately and plated on YEA or YEA containing 0.1% 5-fluoroorotic acid (5-FOA). Colonies were counted after 4 days incubation at 30&#176;C. Mutation rates were calculated by the method of median <xref ref-type="bibr" rid="pgen.1002407-LeaDaC1">[43]</xref>. Genomic DNA from a single 5-FOA resistant colony was isolated and the <italic>ura5-ura4</italic> construct was amplified by PCR to be sequenced.</p><p>Directional 2-D gel analysis was performed as described previously <xref ref-type="bibr" rid="pgen.1002407-Friedman1">[22]</xref> with modifications. Genomic DNA was extracted and digested with <italic>Hind</italic>III and <italic>Blp</italic>I as described in <xref ref-type="bibr" rid="pgen.1002407-Arcangioli1">[44]</xref>. After the first dimension electrophoresis, DNA was digested with <italic>Spe</italic>I in a gel slice and subjected to the second dimension electrophoresis. Replication intermediates were detected by Southern blot.</p><p>Genomic DNA was extracted from exponentially growing cells and purified by Qiagen genomic-tip 100/G. 5 &#181;g of undigested or <italic>Eco</italic>RI digested DNA was incubated in 0.3 M NaOH at 55&#176;C for 2 hours and subjected to 1% alkaline agarose gel electrophoresis <xref ref-type="bibr" rid="pgen.1002407-NickMcElhinny3">[15]</xref>. Gels were neutralized and stained with ethidium bromide, followed by Southern blot.</p><p>Southern blotting was performed according to <xref ref-type="bibr" rid="pgen.1002407-Lambert1">[45]</xref>. DNA fragments of interest were amplified by PCR from <italic>Schizo. pombe</italic> genomic DNA and used as templates to obtain labeled probes. Radioactive nucleotides were incorporated into DNA using Ready-To-Go DNA Labeling beads (GE Healthcare) or strand specific primers and TaKaRa Ex Taq (TAKARA BIO).</p>