<p>Institutional review boards (IRB) at National Cancer Institute, National Institutes of Health and participating institutes approved the study protocols. Written informed consent was obtained from all study participants and/or their legal guardians.</p><p>The study group includes 674 HIV-1 seroconverter European Americans, 669 seronegatives enrolled in the following natural history HIV-1 cohort studies: Multicenter AIDS Cohort Study (MACS) <xref ref-type="bibr" rid="pgen.1002328-Phair1">[59]</xref>, the San Francisco City Clinic Cohort Study (SFCCC) <xref ref-type="bibr" rid="pgen.1002328-Buchbinder1">[60]</xref>, AIDS Link to the Intravenous Experience (ALIVE) <xref ref-type="bibr" rid="pgen.1002328-Vlahov1">[61]</xref>, Hemophilia Growth and Development Study (HGDS) <xref ref-type="bibr" rid="pgen.1002328-Hilgartner1">[62]</xref>, and the Multicenter Hemophilia Cohort Study (MHCS) <xref ref-type="bibr" rid="pgen.1002328-Goedert1">[63]</xref>. Seroconversion date was estimated as the midpoint between the last seronegative and the first seropositive HIV-1 antibody test date (mean interval 0.79 years, range 0.07&#8211;3.0 years). The censoring date was the earliest of the date of the last recorded visit, or July 31, 1997 for the ALIVE cohort, or December 31, 1995 for all other cohorts, to avoid the confounding effect of highly active anti-retroviral therapy (HAART). A later censoring date was used for ALIVE cohort because few ALIVE participants received HAART prior to July 31, 1997 <xref ref-type="bibr" rid="pgen.1002328-Celentano1">[64]</xref>.</p><p>A panel of 72 EA and 72 AA samples representing extreme phenotypes for infection and progression (rapid progression, long term non-progression, and infection resistance) were used for SNP identification. The 5&#8242; and 3&#8242; untranslated (UTR) and coding regions of the <italic>CCR3</italic>, <italic>CCR8</italic>, <italic>CCRL2</italic>, and <italic>CXCR6</italic> genes were PCR amplified by overlapping primer sets (<xref ref-type="supplementary-material" rid="pgen.1002328.s007">Table S2</xref>). PCR products were resequenced by BigDye terminator (Applied Biosystems, Foster City, CA). We did not sequence or genotyping SNPs in <italic>CCR1</italic>, <italic>CCXCR1</italic>, <italic>CCR9</italic> and <italic>CCR4</italic> as they are not recognized as HIV-1 coreceptors <xref ref-type="bibr" rid="pgen.1002328-Rucker1">[26]</xref>, <xref ref-type="bibr" rid="pgen.1002328-Choe1">[33]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1002328-Nedellec1">[36]</xref> (reviewed by <xref ref-type="bibr" rid="pgen.1002328-Berger1">[11]</xref>, <xref ref-type="bibr" rid="pgen.1002328-Lusso1">[37]</xref>).</p><p>Genotyping was done using PCR-restriction fragment length polymorphism (RFLP) or TaqMan assays. PCR primer sequences, TaqMan probes and primers, PCR conditions, and restriction enzymes used to genotype each variant are listed in <xref ref-type="supplementary-material" rid="pgen.1002328.s006">Table S1</xref>. Briefly, PCR was carried out with 35 cycles of denaturing at 94&#176;C for 30 s, annealing at 54&#8211;60&#176;C for 30 s and extension at 72&#176;C for 45 s. TaqMan assays were performed according to the manufacturer's manual (Applied Biosystems, Foster City, CA). The <italic>CX3CR1</italic> variants V249I and T280M were typed as previously reported <xref ref-type="bibr" rid="pgen.1002328-Faure1">[65]</xref>.</p><p>Kaplan-Meier survival statistics and the Cox proportional hazards (PH) model (Cox PH model) were used to assess the effects of genetic variants on the time of progression from HIV-1 infection to AIDS (1987 CDC definition) <xref ref-type="bibr" rid="pgen.1002328-Centers1">[66]</xref>, using PROC PHREG and LIFETEST of SAS version 9.13 (SAS, Cary, North Carolina). For SNPs that showed significant association with AIDS development, explanatory analyses were performed for their specific impact on the AIDS-defining diseases Pneumocystis pneumonia (PCP), Kaposi's sarcoma, microbacterial avium infection, and cytomegalovirus. The relative hazard (RH) and significance of associations were determined using a Cox PH model without or with adjustment for confounding genetic factors not on chromosome 3: for EA <italic>HLA-B*27</italic> and <italic>HLA-B*57</italic>, <italic>HLA-B*35Px</italic>, and <italic>HLA</italic> Class I homozygosity <xref ref-type="bibr" rid="pgen.1002328-Carrington1">[1]</xref>, <xref ref-type="bibr" rid="pgen.1002328-Kaslow1">[67]</xref>, <xref ref-type="bibr" rid="pgen.1002328-Gao1">[68]</xref>; for AA <italic>CCRL5</italic>-In1.1, <italic>HLA-B*57</italic> and <italic>HLA</italic> Class I homozygosity <xref ref-type="bibr" rid="pgen.1002328-Carrington1">[1]</xref>, <xref ref-type="bibr" rid="pgen.1002328-An3">[19]</xref>, <xref ref-type="bibr" rid="pgen.1002328-Kaslow1">[67]</xref>. <italic>CCR2-64I</italic>, <italic>CCR5</italic>-&#916;32 and <italic>CCR5</italic>-2459 were also included as covariates in the adjusted multivariable regression analysis. <italic>CCR5</italic> promoter haplotypes (P1, P2, and P4) are tagged by SNPs <italic>CCR5</italic>-2459 (rs1799987) and rs2734648 <xref ref-type="bibr" rid="pgen.1002328-Carrington2">[43]</xref>. A visual inspection of the data with Kaplan-Meier survival curves was performed to determine the genetic models to be used in the Cox PH regression model. A dominant genetic model was tested for all genetic factors in this study, except for <italic>CCR5</italic>-2459 (recessive) <xref ref-type="bibr" rid="pgen.1002328-Martin1">[15]</xref>, <xref ref-type="bibr" rid="pgen.1002328-McDermott1">[16]</xref>. Participants were stratified by sex and by age at seroconversion: 0&#8211;20, &gt;20&#8211;40, and over 40 years.</p><p>To determine the best explanatory set of genetic variants, while minimizing the number of comparisons in model selection, We used StepAIC procedure to build Cox proportional hazards models for the AIDS-1987 phenotype based on stepwise regression, Akaike information criteria (AIC), and the best subset selection <xref ref-type="bibr" rid="pgen.1002328-Shtatland1">[69]</xref>. Here we used PROC PHREG in SAS software with SLENTRY&#8202;=&#8202;0.99 and SLSTAY&#8202;=&#8202;0.995 (values chosen close to 1 to generate a sequence of models from null models to full model ordered by AIC) <xref ref-type="bibr" rid="pgen.1002328-Harrell1">[48]</xref>. Model uncertainty caused by including large number of variables can be estimated by shrinkage (LR-p)/LR, where <italic>LR</italic> denotes the likelihood ratio <italic>&#967;</italic>
<sup>2</sup> and <italic>p</italic> denotes the number of the predictors in the final model. A shrinkage below 0.85 raises concern of overfitting <xref ref-type="bibr" rid="pgen.1002328-Harrell1">[48]</xref>. It is recommended that no more than m (the number of uncensored event)/10 predictor degree of freedom p (number of parameters) should be examined to fit a multiple regression model. As in our sample, there were 194 events without censoring, we expect that fitting with &lt;19 variables would be appropriate <xref ref-type="bibr" rid="pgen.1002328-Harrell1">[48]</xref>.</p><p>To control for potential population stratification, we adjusted the regression analysis with eigenvector values <xref ref-type="bibr" rid="pgen.1002328-Price1">[70]</xref>. Eigenvector values were obtained by performing principal component analysis of 700,022 SNPs from a previous GWAS study carried out in the same samples <xref ref-type="bibr" rid="pgen.1002328-Troyer1">[71]</xref>. In the Cox regression analysis, eigenvector values for the top two principle components were included as covariates. The genomic inflation factor in this seroconverter population showed minimal systematic overall bias due to population structure in regards to disease progression phenotype as it was quite close to 1.0 (&#955;&#8202;=&#8202;1.01) (expected under no population stratification) <xref ref-type="bibr" rid="pgen.1002328-Troyer1">[71]</xref>.</p><p>We further corrected for multiple comparisons by counting 7 chemokine receptor variants tested using a Bonferroni step-down (Holm) correction method <xref ref-type="bibr" rid="pgen.1002328-Holm1">[47]</xref>, as implemented in the MULTITEST procedure in SAS.</p><p>The &#8220;generalized&#8221; R<sup>2</sup> statistic in Cox model is based on the likelihood-ratio statistic (LRT) for testing the global null hypothesis <xref ref-type="bibr" rid="pgen.1002328-Allison1">[72]</xref>. The formula is given as: R<sup>2</sup>&#8202;=&#8202;1&#8722;e<sup>&#8722;(LRT/n)</sup>, where LRT&#8202;=&#8202;&#8722;2logL(0)&#8722;[&#8722;2logL(p)], n is the total sample size, logL(0) is the log-likelihood for a null model with no covariates, and logL(p) is the log-likelihood for the fitted model with p covariates.</p><p>We quantified LD between all pairs of biallelic SNPs using the absolute unsigned value of Lewontin's D&#8242; statistic <xref ref-type="bibr" rid="pgen.1002328-Lewontin1">[73]</xref>. P values represent significance of departure from the null hypothesis that the pair is in equilibrium. All <italic>P</italic> values are two-tailed. Haploview was used for the LD plots.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Three-dimensional models of the two CCRL2 proteins with 167F or 167Y were constructed using the PROTINFO structure prediction server (<ext-link ext-link-type="uri" ns0:href="http://www.protinfo.compbio.washington.edu">http://www.protinfo.compbio.washington.edu</ext-link>), using the comparative modeling protocol <xref ref-type="bibr" rid="pgen.1002328-Hung1">[74]</xref>, <xref ref-type="bibr" rid="pgen.1002328-Hung2">[75]</xref>. The detailed modeling method is presented in <xref ref-type="supplementary-material" rid="pgen.1002328.s008">Text S1</xref>. The CCRL2 TM topology was established using ConPred II (<ext-link ext-link-type="uri" ns0:href="http://bioinfo.si.hirosaki-u.ac.jp/~ConPred2/">http://bioinfo.si.hirosaki-u.ac.jp/~ConPred2/</ext-link>), a predication program based on consensus results of several prediction methods including TMpred, TMAP,TMHMM, HMTOP and MEMSAT <xref ref-type="bibr" rid="pgen.1002328-Arai1">[50]</xref>.</p><p>cDNAs coding for the full-length open reading frame (ORF) of human CCRL2 carrying 167Y or 167F were PCR amplified from two individuals with the respective homozygous genotype with proofreading DNA polymerase pfu (Stratage, La Jolla, CA). After confirmation of sequence accuracy, they were ligated into the pcDNA3 (Invitrogen, Gaithersburg, MD). Human embryonic epithelial cells line HEK293 was transfected with the constructs of CCRL2-167F or CCRL2-167Y. Stable transfectants were selected by culturing the cells in 800 &#181;g/ml G418. Once the stable cell lines were established, they were examined for chemotactic responses to chemokines.</p><p>Migration of CCRL2-167F and CCRL2-167Y transfected HEK293 cells was assessed using a 48-well microchemotaxis chamber technique. They were examined for chemotactic responses to the following chemokines: CCL5, CCL3, CCL4, CCL2, CCL8, CCL7, CCL11, CXCL12, CCL21, CCL20, and CXCL10. The cells were also tested for migration in response to chemotactic peptides using formayl peptide receptors, including W peptide and MMK-1 (<xref ref-type="supplementary-material" rid="pgen.1002328.s008">Text S1</xref>).</p><p>A cDNA coding for <sup>169</sup>Y-CCR3 was created from the wild-type human <sup>169</sup>F-CCR3 ligated into the pcDNA3, which were used to transfect HEK293 cells (<xref ref-type="supplementary-material" rid="pgen.1002328.s008">Text S1</xref>).</p>